diff --git a/packages/lakka/retroarch_base/retroarch/package.mk b/packages/lakka/retroarch_base/retroarch/package.mk
index ff3d733e4c..25ef805269 100644
--- a/packages/lakka/retroarch_base/retroarch/package.mk
+++ b/packages/lakka/retroarch_base/retroarch/package.mk
@@ -21,7 +21,7 @@ PKG_CONFIGURE_OPTS_TARGET="--disable-vg \
                            --datarootdir=${SYSROOT_PREFIX}/usr/share" # don't use host /usr/share!
 
 PKG_MAKE_OPTS_TARGET="V=1 \
-                      HAVE_LAKKA=1 \
+                      HAVE_LAKKA=0 \
                       HAVE_LAKKA_PROJECT="${DEVICE:-${PROJECT}}.${ARCH}" \
                       HAVE_LAKKA_SERVER="${LAKKA_UPDATE_SERVER_URL}" \
                       HAVE_CHEEVOS=1 \
