diff --git a/packages/lakka/libretro_cores/scummvm_mainline/package.mk b/packages/lakka/libretro_cores/scummvm_mainline/package.mk
new file mode 100644
index 0000000000..4aee5598ae
--- /dev/null
+++ b/packages/lakka/libretro_cores/scummvm_mainline/package.mk
@@ -0,0 +1,30 @@
+
+PKG_NAME="scummvm_mainline"
+PKG_VERSION="92894c3443bc79aa4506f9aef82fea83ab9f7204"
+PKG_LICENSE="GPLv3"
+PKG_SITE="https://github.com/spleen1981/scummvm-mainline-libretro"
+PKG_URL="$PKG_SITE.git"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_SHORTDESC="Script Creation Utility for Maniac Mansion Virtual Machine"
+PKG_LONGDESC="ScummVM is a program which allows you to run certain classic graphical point-and-click adventure games, provided you already have their data files."
+PKG_TOOLCHAIN="make"
+PKG_LR_UPDATE_TAG="yes"
+
+pre_make_target() {
+
+  if [ "${DEVICE}" = "OdroidGoAdvance" ]; then
+    PKG_MAKE_OPTS_TARGET+=" platform=oga_a35_neon_hardfloat"
+  else
+    PKG_MAKE_OPTS_TARGET+=" platform=$TARGET_NAME"
+  fi
+}
+
+makeinstall_target() {
+  mkdir -p ${INSTALL}/usr/lib/libretro
+  cp -v ${PKG_BUILD}/scummvm_mainline_libretro.so ${INSTALL}/usr/lib/libretro/
+  cp -v ${PKG_BUILD}/scummvm_mainline_libretro.info ${INSTALL}/usr/lib/libretro/
+
+  # unpack files to retroarch-system folder and create basic ini file
+  mkdir -p ${INSTALL}/usr/share/retroarch-system
+  unzip ${PKG_BUILD}/scummvm.zip -d ${INSTALL}/usr/share/retroarch-system
+}
