diff --git a/packages/devel/libcec/package.mk b/packages/devel/libcec/package.mk
index f328614a41..5d079fe4ab 100644
--- a/packages/devel/libcec/package.mk
+++ b/packages/devel/libcec/package.mk
@@ -3,8 +3,8 @@
 # Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="libcec"
-PKG_VERSION="6.0.2"
-PKG_SHA256="090696d7a4fb772d7acebbb06f91ab92e025531c7c91824046b9e4e71ecb3377"
+PKG_VERSION="4.0.7"
+PKG_SHA256="bcd92c376993a5721d346edcc09eb17289451f9156b1d1d113c9663c2046315a"
 PKG_LICENSE="GPL"
 PKG_SITE="http://libcec.pulse-eight.com/"
 PKG_URL="https://github.com/Pulse-Eight/libcec/archive/libcec-${PKG_VERSION}.tar.gz"
diff --git a/packages/lakka/libretro_cores/scummvm/package.mk b/packages/lakka/libretro_cores/scummvm/package.mk
index ca12febd96..8fd0a44977 100644
--- a/packages/lakka/libretro_cores/scummvm/package.mk
+++ b/packages/lakka/libretro_cores/scummvm/package.mk
@@ -3,18 +3,19 @@ PKG_VERSION="e1a7e2931877937a04e55f45a9eec6e6f7d471ff"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/scummvm"
 PKG_URL="${PKG_SITE}.git"
-PKG_DEPENDS_TARGET="toolchain curl fluidsynth flac libvorbis zlib faad2 freetype"
+PKG_DEPENDS_TARGET="toolchain curl zlib freetype"
 PKG_SHORTDESC="The ScummVM adventure game engine ported to libretro."
 PKG_LONGDESC="ScummVM is a program which allows you to run certain classic graphical point-and-click adventure games, provided you already have their data files."
 PKG_TOOLCHAIN="make"
 
 PKG_MAKE_OPTS_TARGET="USE_CLOUD=1 \
-                      USE_SYSTEM_fluidsynt=1 \
-                      USE_SYSTEM_FLAC=1 \
-                      USE_SYSTEM_vorbis=1 \
+                      USE_SYSTEM_fluidsynt=0 \
+                      USE_SYSTEM_FLAC=0 \
+                      USE_SYSTEM_vorbis=0 \
                       USE_SYSTEM_z=1 \
-                      USE_SYSTEM_faad=1 \
-                      USE_SYSTEM_freetype=1"
+                      USE_SYSTEM_faad=0 \
+                      USE_SYSTEM_freetype=1 \
+                      FORCE_OPENGLES2=1"
 
 pre_make_target() {
   if [ "${DEVICE}" = "OdroidGoAdvance" ]; then
