diff --git a/packages/lakka/libretro_cores/scummvm/package.mk b/packages/lakka/libretro_cores/scummvm/package.mk
index 16b2193106..556e07273c 100644
--- a/packages/lakka/libretro_cores/scummvm/package.mk
+++ b/packages/lakka/libretro_cores/scummvm/package.mk
@@ -3,18 +3,19 @@ PKG_VERSION="c9e94a9d7417119c94f4a3b21f4284c3dda926e2"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/scummvm"
 PKG_URL="${PKG_SITE}.git"
-PKG_DEPENDS_TARGET="toolchain curl fluidsynth flac libvorbis zlib faad2 freetype"
+PKG_DEPENDS_TARGET="toolchain curl zlib freetype"
 PKG_LONGDESC="ScummVM with libretro backend."
 PKG_TOOLCHAIN="make"
 
 PKG_MAKE_OPTS_TARGET="all \
                       USE_CLOUD=1 \
-                      USE_SYSTEM_fluidsynth=1 \
-                      USE_SYSTEM_FLAC=1 \
-                      USE_SYSTEM_vorbis=1 \
+                      USE_SYSTEM_fluidsynth=0 \
+                      USE_SYSTEM_FLAC=0 \
+                      USE_SYSTEM_vorbis=0 \
                       USE_SYSTEM_z=1 \
-                      USE_SYSTEM_faad=1 \
-                      USE_SYSTEM_freetype=1"
+                      USE_SYSTEM_faad=0 \
+                      USE_SYSTEM_freetype=1 \
+                      FORCE_OPENGLES2=1"
 
 pre_make_target() {
   PKG_MAKE_OPTS_TARGET+=" -C ${PKG_BUILD}/backends/platform/libretro"
