diff --git a/packages/lakka/retroarch_base/retroarch/package.mk b/packages/lakka/retroarch_base/retroarch/package.mk
index 4bd4c11efa..8b9a0ed10a 100644
--- a/packages/lakka/retroarch_base/retroarch/package.mk
+++ b/packages/lakka/retroarch_base/retroarch/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="retroarch"
-PKG_VERSION="d9b90e218238b183d6fbaad2c8bfc9fcd8cb0190"
+PKG_VERSION="ad8975cb5a0fe45be43438bdbd6c3d745653dd02"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/RetroArch"
 PKG_URL="${PKG_SITE}.git"
