diff --git a/packages/addons/addon-depends/libzip/package.mk b/packages/addons/addon-depends/libzip/package.mk
index a8afc31861..d1d9a90157 100644
--- a/packages/addons/addon-depends/libzip/package.mk
+++ b/packages/addons/addon-depends/libzip/package.mk
@@ -3,8 +3,8 @@
 # Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="libzip"
-PKG_VERSION="1.8.0"
-PKG_SHA256="f0763bda24ba947e80430be787c4b068d8b6aa6027a26a19923f0acfa3dac97e"
+PKG_VERSION="1.9.2"
+PKG_SHA256="c93e9852b7b2dc931197831438fee5295976ee0ba24f8524a8907be5c2ba5937"
 PKG_LICENSE="GPL"
 PKG_SITE="https://libzip.org/"
 PKG_URL="https://libzip.org/download/${PKG_NAME}-${PKG_VERSION}.tar.xz"
diff --git a/packages/addons/addon-depends/libzip/patches/libzip-001-fix-typo-which-prevents-expansion-of-libdir-in-pkgconfig.patch b/packages/addons/addon-depends/libzip/patches/libzip-001-fix-typo-which-prevents-expansion-of-libdir-in-pkgconfig.patch
deleted file mode 100644
index fa319b55b0..0000000000
--- a/packages/addons/addon-depends/libzip/patches/libzip-001-fix-typo-which-prevents-expansion-of-libdir-in-pkgconfig.patch
+++ /dev/null
@@ -1,105 +0,0 @@
-From ca61524e05d785f0d14f40802c6a1a54a46b74bb Mon Sep 17 00:00:00 2001
-From: rezso <rezso@rezso.net>
-Date: Sat, 19 Jun 2021 01:50:26 +0200
-Subject: [PATCH] CMakeLists.txt: add missing regex for zstd
-
-Currently cmake provides an pc file with:
-Libs.private:  -lbz2 -llzma -lZstd::Zstd -lz
----
- CMakeLists.txt | 1 +
- 1 file changed, 1 insertion(+)
-
-diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 93b46a61..defc8745 100644
---- a/CMakeLists.txt
-+++ b/CMakeLists.txt
-@@ -299,6 +299,7 @@ foreach(LIB ${LIBS_PRIVATE})
- endforeach()
- string(REGEX REPLACE "-lBZip2::BZip2" "-lbz2" LIBS ${LIBS})
- string(REGEX REPLACE "-lLibLZMA::LibLZMA" "-llzma" LIBS ${LIBS})
-+string(REGEX REPLACE "-lZstd::Zstd" "-lzstd" LIBS ${LIBS})
- string(REGEX REPLACE "-lOpenSSL::Crypto" "-lssl -lcrypto" LIBS ${LIBS})
- string(REGEX REPLACE "-lZLIB::ZLIB" "-lz" LIBS ${LIBS})
- string(REGEX REPLACE "-lGnuTLS::GnuTLS" "-lgnutls" LIBS ${LIBS})
-
-From c4af9182efd9cbb127bb43486b55d9ddf4a521ca Mon Sep 17 00:00:00 2001
-From: Thomas Klausner <tk@giga.or.at>
-Date: Mon, 28 Jun 2021 11:52:15 +0200
-Subject: [PATCH] Try having relative paths in pkg-config file.
-
-Addresses #248
----
- CMakeLists.txt | 11 ++++++-----
- libzip.pc.in   |  7 +++----
- 2 files changed, 9 insertions(+), 9 deletions(-)
-
-diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 8611a405..93f89df1 100644
---- a/CMakeLists.txt
-+++ b/CMakeLists.txt
-@@ -281,11 +281,12 @@ endif()
- 
- 
- # pkgconfig file
--set(prefix ${CMAKE_INSTALL_PREFIX})
--set(exec_prefix \${prefix})
--SET(bindir ${CMAKE_INSTALL_FULL_BINDIR})
--SET(libdir ${CMAKE_INSTALL_FULL_LIBDIR})
--SET(includedir ${CMAKE_INSTALL_FULL_INCLUDEDIR})
-+file(RELATIVE_PATH pc_relative_bindir ${CMAKE_INSTALL_PREFIX} ${CMAKE_INSTALL_FULL_BINDIR})
-+set(bindir "\${prefix}/${pc_relative_bindir}")
-+file(RELATIVE_PATH pc_relative_libdir ${CMAKE_INSTALL_PREFIX} ${CMAKE_INSTALL_FULL_LIBDIR})
-+set(libdir "\${prefix)/${pc_relative_libdir}")
-+file(RELATIVE_PATH pc_relative_includedir ${CMAKE_INSTALL_PREFIX} ${CMAKE_INSTALL_FULL_INCLUDEDIR})
-+set(includedir "\${prefix}/${pc_relative_includedir}")
- if(CMAKE_SYSTEM_NAME MATCHES BSD)
-   set(PKG_CONFIG_RPATH "-Wl,-R\${libdir}")
- endif(CMAKE_SYSTEM_NAME MATCHES BSD)
-diff --git a/libzip.pc.in b/libzip.pc.in
-index 58b3dca7..d51b0ab6 100644
---- a/libzip.pc.in
-+++ b/libzip.pc.in
-@@ -1,10 +1,10 @@
--prefix=@prefix@
--exec_prefix=@exec_prefix@
-+prefix=@CMAKE_INSTALL_PREFIX@
-+exec_prefix=${prefix}
- bindir=@bindir@
- libdir=@libdir@
- includedir=@includedir@
- 
--zipcmp=@bindir@/zipcmp
-+zipcmp=${bindir}/zipcmp
- 
- Name: libzip
- Description: library for handling zip archives
-@@ -12,4 +12,3 @@ Version: @PROJECT_VERSION@
- Libs: @PKG_CONFIG_RPATH@ -L${libdir} -lzip
- Libs.private: @LIBS@
- Cflags: -I${includedir}
--
-
-From 28951cbf403ea3a788f823377bbc171d1e1359ba Mon Sep 17 00:00:00 2001
-From: Michael Heimpold <mhei@heimpold.de>
-Date: Sat, 3 Jul 2021 00:04:14 +0200
-Subject: [PATCH] Fix typo which prevents expansion of libdir in pkgconfig
- files
-
-Signed-off-by: Michael Heimpold <mhei@heimpold.de>
----
- CMakeLists.txt | 2 +-
- 1 file changed, 1 insertion(+), 1 deletion(-)
-
-diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 93f89df1..2fb87165 100644
---- a/CMakeLists.txt
-+++ b/CMakeLists.txt
-@@ -284,7 +284,7 @@ endif()
- file(RELATIVE_PATH pc_relative_bindir ${CMAKE_INSTALL_PREFIX} ${CMAKE_INSTALL_FULL_BINDIR})
- set(bindir "\${prefix}/${pc_relative_bindir}")
- file(RELATIVE_PATH pc_relative_libdir ${CMAKE_INSTALL_PREFIX} ${CMAKE_INSTALL_FULL_LIBDIR})
--set(libdir "\${prefix)/${pc_relative_libdir}")
-+set(libdir "\${prefix}/${pc_relative_libdir}")
- file(RELATIVE_PATH pc_relative_includedir ${CMAKE_INSTALL_PREFIX} ${CMAKE_INSTALL_FULL_INCLUDEDIR})
- set(includedir "\${prefix}/${pc_relative_includedir}")
- if(CMAKE_SYSTEM_NAME MATCHES BSD)
diff --git a/packages/graphics/libpng/package.mk b/packages/graphics/libpng/package.mk
index a2c373f198..f31068fc14 100644
--- a/packages/graphics/libpng/package.mk
+++ b/packages/graphics/libpng/package.mk
@@ -2,12 +2,8 @@
 # Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
 
 PKG_NAME="libpng"
-PKG_VERSION="1.6.37"
-PKG_SHA256="505e70834d35383537b6491e7ae8641f1a4bed1876dbfe361201fc80868d88ca"
-if [ "${DISTRO}" = "Lakka" ]; then
-  PKG_VERSION="1.7.0beta89"
-  PKG_SHA256="1fad2475a24174f5b4ad237b8b899a2c0583237f108c2288a6e2ac5c3537147a"
-fi
+PKG_VERSION="1.6.38"
+PKG_SHA256="b3683e8b8111ebf6f1ac004ebb6b0c975cd310ec469d98364388e9cedbfa68be"
 PKG_LICENSE="LibPNG2"
 PKG_SITE="http://www.libpng.org/"
 PKG_URL="${SOURCEFORGE_SRC}/libpng/${PKG_NAME}-${PKG_VERSION}.tar.xz"
diff --git a/packages/lakka/libretro_cores/2048/package.mk b/packages/lakka/libretro_cores/2048/package.mk
index 42245fd35c..a5416138c4 100644
--- a/packages/lakka/libretro_cores/2048/package.mk
+++ b/packages/lakka/libretro_cores/2048/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="2048"
-PKG_VERSION="a8899c960c0f8fca24f9c319cb935fe164b97bf9"
+PKG_VERSION="0edef01f4c2dd2cbd0bb66b888ca4b57fad297d1"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/libretro-2048"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/81/package.mk b/packages/lakka/libretro_cores/81/package.mk
index 3d8eebbe1c..91e56f58be 100644
--- a/packages/lakka/libretro_cores/81/package.mk
+++ b/packages/lakka/libretro_cores/81/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="81"
-PKG_VERSION="2e34567a320cba27b9162b1776db4de3cdb7cf03"
+PKG_VERSION="340a51b250fb8fbf1a9e5d3ad3924044250064e0"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/81-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/a5200/package.mk b/packages/lakka/libretro_cores/a5200/package.mk
index 58bff363d6..5e567df8b4 100644
--- a/packages/lakka/libretro_cores/a5200/package.mk
+++ b/packages/lakka/libretro_cores/a5200/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="a5200"
-PKG_VERSION="599a2e53feaf47c920bbb0f1f419cf54af570d58"
+PKG_VERSION="b8f8571eb5c6f484fe6be9a3a895ffb162b08422"
 PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/libretro/a5200"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/atari800/package.mk b/packages/lakka/libretro_cores/atari800/package.mk
index e9edb5a0e1..728f53a735 100644
--- a/packages/lakka/libretro_cores/atari800/package.mk
+++ b/packages/lakka/libretro_cores/atari800/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="atari800"
-PKG_VERSION="beab30e7ea10b7ed14d0514064f47d16f76cd995"
+PKG_VERSION="94033288b026fe699bc50703609807aa8075f4dd"
 PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/libretro/libretro-atari800"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/beetle_lynx/package.mk b/packages/lakka/libretro_cores/beetle_lynx/package.mk
index 766ebe9663..5fcf7112d4 100644
--- a/packages/lakka/libretro_cores/beetle_lynx/package.mk
+++ b/packages/lakka/libretro_cores/beetle_lynx/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="beetle_lynx"
-PKG_VERSION="de0d520d679cb92767876d4e98da908b1ea6a2d6"
+PKG_VERSION="9d5288370a72bbc8bf3d20670d83fb9118a771b0"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/beetle-lynx-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/beetle_ngp/package.mk b/packages/lakka/libretro_cores/beetle_ngp/package.mk
index f206d31a86..18eeb6be5f 100644
--- a/packages/lakka/libretro_cores/beetle_ngp/package.mk
+++ b/packages/lakka/libretro_cores/beetle_ngp/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="beetle_ngp"
-PKG_VERSION="facf8e1f5440c5d289258ee3c483710f3bf916fb"
+PKG_VERSION="00c7cb8ea97ad9a372307405d8abf34e401fec8a"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/beetle-ngp-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/beetle_pce/package.mk b/packages/lakka/libretro_cores/beetle_pce/package.mk
index 4f0235b36e..693b87b342 100644
--- a/packages/lakka/libretro_cores/beetle_pce/package.mk
+++ b/packages/lakka/libretro_cores/beetle_pce/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="beetle_pce"
-PKG_VERSION="ccb9096cdfbb35b0b3a05094b1ff73173cf2144f"
+PKG_VERSION="698bceadb5a3fe158b224e750f0c5016afe9241e"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/beetle-pce-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/beetle_pce_fast/package.mk b/packages/lakka/libretro_cores/beetle_pce_fast/package.mk
index 5aa5788df9..a3b6734099 100644
--- a/packages/lakka/libretro_cores/beetle_pce_fast/package.mk
+++ b/packages/lakka/libretro_cores/beetle_pce_fast/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="beetle_pce_fast"
-PKG_VERSION="e8801687f232a6f8828b3ff5dadbc9fe1b0076fc"
+PKG_VERSION="617a8ddbab19b03f5524d7b59b9a8d25d87420af"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/beetle-pce-fast-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/beetle_pcfx/package.mk b/packages/lakka/libretro_cores/beetle_pcfx/package.mk
index 16f4a88ad8..a8d86616f4 100644
--- a/packages/lakka/libretro_cores/beetle_pcfx/package.mk
+++ b/packages/lakka/libretro_cores/beetle_pcfx/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="beetle_pcfx"
-PKG_VERSION="bfc0954e14b261a04dcf8dbe0df8798f16ae3f3c"
+PKG_VERSION="7c85e91fa88bbb5f43841aaeba7dcad0bd21680f"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/beetle-pcfx-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/beetle_psx/package.mk b/packages/lakka/libretro_cores/beetle_psx/package.mk
index ae2ed5852e..70b01896f4 100644
--- a/packages/lakka/libretro_cores/beetle_psx/package.mk
+++ b/packages/lakka/libretro_cores/beetle_psx/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="beetle_psx"
-PKG_VERSION="88929ae90b4807a41b1b240377ab440e39ecf2cc"
+PKG_VERSION="a87d6f3ed5d467f62179c2be63559b7025d2471d"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/beetle-psx-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/beetle_saturn/package.mk b/packages/lakka/libretro_cores/beetle_saturn/package.mk
index 50c3fbbe65..9d8fb8b43f 100644
--- a/packages/lakka/libretro_cores/beetle_saturn/package.mk
+++ b/packages/lakka/libretro_cores/beetle_saturn/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="beetle_saturn"
-PKG_VERSION="ae30f29e340a00b33e38df85ceaa599151a47cd7"
+PKG_VERSION="054862a4ccb9b2f1bad9e5b075fc3d1116dc8055"
 PKG_ARCH="x86_64"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/beetle-saturn-libretro"
diff --git a/packages/lakka/libretro_cores/beetle_supafaust/package.mk b/packages/lakka/libretro_cores/beetle_supafaust/package.mk
index f8f29b3fe3..89c113189f 100644
--- a/packages/lakka/libretro_cores/beetle_supafaust/package.mk
+++ b/packages/lakka/libretro_cores/beetle_supafaust/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="beetle_supafaust"
-PKG_VERSION="85b5527231a6ad6f9475c15c8ff1b9d16884cd30"
+PKG_VERSION="2f49621be26ecdd31ef247eb083cd2fd1c5e6744"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/supafaust"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/beetle_supergrafx/package.mk b/packages/lakka/libretro_cores/beetle_supergrafx/package.mk
index 6be3c62c9c..6c87c39a50 100644
--- a/packages/lakka/libretro_cores/beetle_supergrafx/package.mk
+++ b/packages/lakka/libretro_cores/beetle_supergrafx/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="beetle_supergrafx"
-PKG_VERSION="59991a98c232b1a8350a9d67ac554c5b22771d3c"
+PKG_VERSION="765fa6ea8b720d294a14e895213d369947d68a39"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/beetle-supergrafx-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/beetle_vb/package.mk b/packages/lakka/libretro_cores/beetle_vb/package.mk
index e361baf601..ee96985dfe 100644
--- a/packages/lakka/libretro_cores/beetle_vb/package.mk
+++ b/packages/lakka/libretro_cores/beetle_vb/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="beetle_vb"
-PKG_VERSION="246555f8ed7e0b9e5748b2ee2ed6743187c61393"
+PKG_VERSION="162918f06d9a705330b2ba128e0d3b65fd1a1bcc"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/beetle-vb-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/beetle_wswan/package.mk b/packages/lakka/libretro_cores/beetle_wswan/package.mk
index fa0a1dbc05..88b900125f 100644
--- a/packages/lakka/libretro_cores/beetle_wswan/package.mk
+++ b/packages/lakka/libretro_cores/beetle_wswan/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="beetle_wswan"
-PKG_VERSION="d1fb3f399a2bc16b9ad0f2e8c8ba9f7051cd26bd"
+PKG_VERSION="3cac80a102938cf8c6ca76c85bbe1932f06704a2"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/beetle-wswan-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/blastem/package.mk b/packages/lakka/libretro_cores/blastem/package.mk
index 311a2acbcc..646a6980a6 100644
--- a/packages/lakka/libretro_cores/blastem/package.mk
+++ b/packages/lakka/libretro_cores/blastem/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="blastem"
-PKG_VERSION="0786858437ed71996f43b7af0fbe627eb88152fc"
+PKG_VERSION="277e4a62668597d4f59cadda1cbafb844f981d45"
 PKG_ARCH="x86_64 i386"
 PKG_LICENSE="GPL3"
 PKG_SITE="https://github.com/libretro/blastem"
diff --git a/packages/lakka/libretro_cores/bluemsx/package.mk b/packages/lakka/libretro_cores/bluemsx/package.mk
index ec8b1bc28a..59be9a8393 100644
--- a/packages/lakka/libretro_cores/bluemsx/package.mk
+++ b/packages/lakka/libretro_cores/bluemsx/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="bluemsx"
-PKG_VERSION="92d0c41b4973854114c7b2d06ab727a266d404c5"
+PKG_VERSION="acf358be18644a9df0ed9602d63c2f73d4fe605a"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/blueMSX-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/bsnes2014/package.mk b/packages/lakka/libretro_cores/bsnes2014/package.mk
index 11e6980bc0..908fa157bc 100644
--- a/packages/lakka/libretro_cores/bsnes2014/package.mk
+++ b/packages/lakka/libretro_cores/bsnes2014/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="bsnes2014"
-PKG_VERSION="a8f232a38bc54a6b1871d43504a4c098cb5c0709"
+PKG_VERSION="78dc66f8c09dc0117d55ee4249186674385386e5"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/bsnes2014"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/bsnes_hd/package.mk b/packages/lakka/libretro_cores/bsnes_hd/package.mk
index 7ec5ed4892..1cc038ff6d 100644
--- a/packages/lakka/libretro_cores/bsnes_hd/package.mk
+++ b/packages/lakka/libretro_cores/bsnes_hd/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="bsnes_hd"
-PKG_VERSION="65f24e56c37f46bb752190024bd4058e64ad77d1"
+PKG_VERSION="04821703aefdc909a4fd66d168433fcac06c2ba7"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/DerKoun/bsnes-hd"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/bsnes_mercury/package.mk b/packages/lakka/libretro_cores/bsnes_mercury/package.mk
index dbba5c0b2d..733a8caa02 100644
--- a/packages/lakka/libretro_cores/bsnes_mercury/package.mk
+++ b/packages/lakka/libretro_cores/bsnes_mercury/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="bsnes_mercury"
-PKG_VERSION="4ba6d8d88e57d3193d95e1bcf39e8d31121f76d4"
+PKG_VERSION="fb9a41fe9bc230a07c4506cad3cbf21d3fa635b4"
 PKG_ARCH="x86_64 i386"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/bsnes-mercury"
diff --git a/packages/lakka/libretro_cores/cap32/package.mk b/packages/lakka/libretro_cores/cap32/package.mk
index 25b5b414ac..78f33a3393 100644
--- a/packages/lakka/libretro_cores/cap32/package.mk
+++ b/packages/lakka/libretro_cores/cap32/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="cap32"
-PKG_VERSION="a1b2329416b1e3f32e0859dd8d3c71cb8c3e1c08"
+PKG_VERSION="73dc9ce0b205f797c174706732d6161d634a224c"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/libretro-cap32"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/chailove/package.mk b/packages/lakka/libretro_cores/chailove/package.mk
index dd3b7f630a..9aea8520c7 100644
--- a/packages/lakka/libretro_cores/chailove/package.mk
+++ b/packages/lakka/libretro_cores/chailove/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="chailove"
-PKG_VERSION="0fedc82f49b006d30c3fa71110908e84acaf8b75"
+PKG_VERSION="9677b142ab9562cb278e87d7475c55fe0b9a4333"
 PKG_LICENSE="MIT"
 PKG_SITE="https://github.com/libretro/libretro-chailove"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/citra/package.mk b/packages/lakka/libretro_cores/citra/package.mk
index 848ea4ccf8..e165347e7a 100644
--- a/packages/lakka/libretro_cores/citra/package.mk
+++ b/packages/lakka/libretro_cores/citra/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="citra"
-PKG_VERSION="44e01f99016008eff18bc7a28234d1098382358d"
+PKG_VERSION="aa165606f1e2d74e0edb9b25b20f1f566aa91e35"
 PKG_ARCH="x86_64"
 PKG_LICENSE="GPLv2+"
 PKG_SITE="https://github.com/libretro/citra"
diff --git a/packages/lakka/libretro_cores/crocods/package.mk b/packages/lakka/libretro_cores/crocods/package.mk
index 2c69bc58b8..d7117cbbe1 100644
--- a/packages/lakka/libretro_cores/crocods/package.mk
+++ b/packages/lakka/libretro_cores/crocods/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="crocods"
-PKG_VERSION="86eeccfb68ad6a1cb39ef5b008902f6636d7d194"
+PKG_VERSION="f1b4b91291ba1e8e7c0be02269cd0d75c7fa71b9"
 PKG_LICENSE="MIT"
 PKG_SITE="https://github.com/libretro/libretro-crocods"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/daphne/package.mk b/packages/lakka/libretro_cores/daphne/package.mk
index 86c0cbe51f..e12de1b98d 100644
--- a/packages/lakka/libretro_cores/daphne/package.mk
+++ b/packages/lakka/libretro_cores/daphne/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="daphne"
-PKG_VERSION="0a7e6f0fda1348144369ce0a889876df60263e8f"
+PKG_VERSION="b5481bab34a51369b6749cd95f5f889e43aaa23f"
 PKG_LICENSE="GPLv2+"
 PKG_SITE="https://github.com/libretro/daphne"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/desmume/package.mk b/packages/lakka/libretro_cores/desmume/package.mk
index ae41c23622..c8ae4b1927 100644
--- a/packages/lakka/libretro_cores/desmume/package.mk
+++ b/packages/lakka/libretro_cores/desmume/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="desmume"
-PKG_VERSION="5d0ae2be2c9fb6362af528b3722e81323318eb9f"
+PKG_VERSION="fbd368c8109f95650e1f81bca1facd6d4d8687d7"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/desmume"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/dolphin/package.mk b/packages/lakka/libretro_cores/dolphin/package.mk
index c494c26597..c9e83e72a2 100644
--- a/packages/lakka/libretro_cores/dolphin/package.mk
+++ b/packages/lakka/libretro_cores/dolphin/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="dolphin"
-PKG_VERSION="6a0b6ee8a4d5363e669f5faf43abc8f17e4278a8"
+PKG_VERSION="44394cee28d44c70debdb176ad2f2f5e38e9f04c"
 PKG_ARCH="x86_64 aarch64"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/dolphin"
diff --git a/packages/lakka/libretro_cores/dosbox/package.mk b/packages/lakka/libretro_cores/dosbox/package.mk
index c369b44a57..9cb762105d 100644
--- a/packages/lakka/libretro_cores/dosbox/package.mk
+++ b/packages/lakka/libretro_cores/dosbox/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="dosbox"
-PKG_VERSION="74cd17ed0ff810ff78cb8c1f1e45513bfe8a0f32"
+PKG_VERSION="b7b24262c282c0caef2368c87323ff8c381b3102"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/dosbox-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/dosbox_pure/package.mk b/packages/lakka/libretro_cores/dosbox_pure/package.mk
index 276bc4f06e..b77ee93b2e 100644
--- a/packages/lakka/libretro_cores/dosbox_pure/package.mk
+++ b/packages/lakka/libretro_cores/dosbox_pure/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="dosbox_pure"
-PKG_VERSION="d22a43d80a4dd17d64bcbd3977c85d06ac2c0dfe"
+PKG_VERSION="4fdb557e415698aae5bd90b076f76437f5e9b0b4"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/dosbox-pure"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/dosbox_pure/patches/dosbox_pure-Makefile-optimization_and_libs.patch b/packages/lakka/libretro_cores/dosbox_pure/patches/dosbox_pure-Makefile-optimization_and_libs.patch
index 6b1b691237..ed49ff491d 100644
--- a/packages/lakka/libretro_cores/dosbox_pure/patches/dosbox_pure-Makefile-optimization_and_libs.patch
+++ b/packages/lakka/libretro_cores/dosbox_pure/patches/dosbox_pure-Makefile-optimization_and_libs.patch
@@ -1,8 +1,8 @@
 diff --git a/Makefile b/Makefile
-index 8bdbfa7..cb9674f 100644
+index f0acd51..f118214 100644
 --- a/Makefile
 +++ b/Makefile
-@@ -192,9 +194,9 @@ else
+@@ -201,9 +201,9 @@ else
    ifeq ($(platform),vita)
      CFLAGS   := -DNDEBUG -O3 -fno-ident -fno-partial-inlining
    else
@@ -13,23 +13,4 @@ index 8bdbfa7..cb9674f 100644
 +  LDFLAGS  += -O3 -fno-ident
  endif
  
- CFLAGS  += $(CPUFLAGS) -std=gnu++11 -fomit-frame-pointer -fno-exceptions -fno-non-call-exceptions -Wno-address-of-packed-member -Wno-format -Wno-switch -Wno-psabi
-@@ -209,7 +211,8 @@ CFLAGS  += -D__LIBRETRO__ -Iinclude -D_FILE_OFFSET_BITS=64
- CFLAGS  += $(COMMONFLAGS)
- #CFLAGS  += -fdata-sections #saves around 32 bytes on most platforms but wrongfully adds up to 60MB on msys2
- 
--LDFLAGS += $(CPUFLAGS) -lpthread -shared
-+LDFLAGS += $(CPUFLAGS) -shared
-+LDLIBS  := -lpthread
- #LDFLAGS += -static-libstdc++ -static-libgcc #adds 1MB to output and still dynamically links against libc and libm
- 
- .PHONY: all clean
-@@ -234,7 +237,7 @@ ifeq ($(STATIC_LINKING), 1)
- 	$(AR) rcs $@ $^
- else
- 	$(info Linking $@ ...)
--	$(CXX) $(LDFLAGS) -o $@ $^
-+	$(CXX) $(LDFLAGS) -o $@ $^ $(LDLIBS)
- ifneq ($(BUILD),DEBUG)
- 	@-/opt/gcw0-toolchain/usr/mipsel-gcw0-linux-uclibc/bin/strip --strip-all $@ $(PIPETONULL);true # gcw0
- 	@-strip --strip-all $@ $(PIPETONULL);true # others
+ CFLAGS  += $(CPUFLAGS) -std=gnu++11 -fomit-frame-pointer -fexceptions -Wno-address-of-packed-member -Wno-format -Wno-switch -Wno-psabi
diff --git a/packages/lakka/libretro_cores/dosbox_svn/package.mk b/packages/lakka/libretro_cores/dosbox_svn/package.mk
index 720c803193..19950c527c 100644
--- a/packages/lakka/libretro_cores/dosbox_svn/package.mk
+++ b/packages/lakka/libretro_cores/dosbox_svn/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="dosbox_svn"
-PKG_VERSION="4f2b9ca30cd9487ac8610762552c553b0c83892d"
+PKG_VERSION="c23be7769518d753378307996de35e204d188c63"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/dosbox-svn"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/ecwolf/package.mk b/packages/lakka/libretro_cores/ecwolf/package.mk
index ce36539932..907bc32c2c 100644
--- a/packages/lakka/libretro_cores/ecwolf/package.mk
+++ b/packages/lakka/libretro_cores/ecwolf/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="ecwolf"
-PKG_VERSION="e86cd4c4bed96a160e1a6416d387a9aae0b38402"
+PKG_VERSION="f098da0d003c4780adf6a9503801081f1f25cc27"
 PKG_LICENSE="Unknown"
 PKG_SITE="https://github.com/libretro/ecwolf"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/fbneo/package.mk b/packages/lakka/libretro_cores/fbneo/package.mk
index b87048fba2..f7326d4188 100644
--- a/packages/lakka/libretro_cores/fbneo/package.mk
+++ b/packages/lakka/libretro_cores/fbneo/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="fbneo"
-PKG_VERSION="3d4c2e73c380b57ceba26a68071a635087b91771"
+PKG_VERSION="87423da080a469e99633160cbe4bd35b5ca94b41"
 PKG_LICENSE="Non-commercial"
 PKG_SITE="https://github.com/libretro/fbneo"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/fceumm/package.mk b/packages/lakka/libretro_cores/fceumm/package.mk
index fc2c928000..33f52ca8ff 100644
--- a/packages/lakka/libretro_cores/fceumm/package.mk
+++ b/packages/lakka/libretro_cores/fceumm/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="fceumm"
-PKG_VERSION="b3c35b6515b2b6a789c589f976a4a323ebebe3eb"
+PKG_VERSION="8c3f690e61a1d65dfb25510426ae88eeae93e1ae"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/libretro-fceumm"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/flycast/package.mk b/packages/lakka/libretro_cores/flycast/package.mk
index e329e21b9c..bd6f70a4c8 100644
--- a/packages/lakka/libretro_cores/flycast/package.mk
+++ b/packages/lakka/libretro_cores/flycast/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="flycast"
-PKG_VERSION="fd50529d864c0af7007f2f283a84bf67851fe0e4"
+PKG_VERSION="8a0323ec087f8bf85a419ac7ced575f6065d5758"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/flyinghead/flycast"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/fmsx/package.mk b/packages/lakka/libretro_cores/fmsx/package.mk
index a0f98efdfe..2d9dcc1ab3 100644
--- a/packages/lakka/libretro_cores/fmsx/package.mk
+++ b/packages/lakka/libretro_cores/fmsx/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="fmsx"
-PKG_VERSION="11fa9f3c08cde567394c41320ca76798c2c64670"
+PKG_VERSION="1360c9ff32b390383567774d01fbe5d6dfcadaa3"
 PKG_LICENSE="Non-commercial"
 PKG_SITE="https://github.com/libretro/fmsx-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/freechaf/package.mk b/packages/lakka/libretro_cores/freechaf/package.mk
index 2f7995ef4c..7f3efa63f1 100644
--- a/packages/lakka/libretro_cores/freechaf/package.mk
+++ b/packages/lakka/libretro_cores/freechaf/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="freechaf"
-PKG_VERSION="bc5a4ee2889930b7b8d1dd390c8c6ba29f7644dd"
+PKG_VERSION="4d1d5cb83b93728a63f03454e472a23055d9bbfc"
 PKG_LICENSE="GPL3"
 PKG_SITE="https://github.com/libretro/FreeChaF"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/freeintv/package.mk b/packages/lakka/libretro_cores/freeintv/package.mk
index 27bd923140..209149852d 100644
--- a/packages/lakka/libretro_cores/freeintv/package.mk
+++ b/packages/lakka/libretro_cores/freeintv/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="freeintv"
-PKG_VERSION="295dd3c9e4b2d4f652f6a6a904afbe90a8187068"
+PKG_VERSION="9a65ec6e31d48ad0dae1f381c1ec61c897f970cb"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/FreeIntv"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/fuse_libretro/package.mk b/packages/lakka/libretro_cores/fuse_libretro/package.mk
index 5c3922b5dc..729d91a1ad 100644
--- a/packages/lakka/libretro_cores/fuse_libretro/package.mk
+++ b/packages/lakka/libretro_cores/fuse_libretro/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="fuse_libretro"
-PKG_VERSION="8b734a975633a0ed21494c2e1a50e7e1de432122"
+PKG_VERSION="3f9344ddf88dfd251d95cffdea615692479e8cc6"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/fuse-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/gambatte/package.mk b/packages/lakka/libretro_cores/gambatte/package.mk
index ea683fc4a8..f39545f19e 100644
--- a/packages/lakka/libretro_cores/gambatte/package.mk
+++ b/packages/lakka/libretro_cores/gambatte/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="gambatte"
-PKG_VERSION="15536214cdce31894d374b2ffa2494543057082b"
+PKG_VERSION="7e02df60048db0898131ea365f387a026e4e648d"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/gambatte-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/gearboy/package.mk b/packages/lakka/libretro_cores/gearboy/package.mk
index e561b56967..e89116f036 100644
--- a/packages/lakka/libretro_cores/gearboy/package.mk
+++ b/packages/lakka/libretro_cores/gearboy/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="gearboy"
-PKG_VERSION="2cb66cdcb05f77147b9a5941e8ab92bceea828ae"
+PKG_VERSION="2c5b8b426332b3da43905ce796b8f7e7abc07444"
 PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/drhelius/Gearboy"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/gearsystem/package.mk b/packages/lakka/libretro_cores/gearsystem/package.mk
index efb7ab2896..71b93ecd07 100644
--- a/packages/lakka/libretro_cores/gearsystem/package.mk
+++ b/packages/lakka/libretro_cores/gearsystem/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="gearsystem"
-PKG_VERSION="a6849bfa6973af6417678d9b1ea893064aa3910c"
+PKG_VERSION="e3ab748c52c6e9bfb8906ba06723ba18ef7aa88a"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/drhelius/Gearsystem"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/genesis_plus_gx/package.mk b/packages/lakka/libretro_cores/genesis_plus_gx/package.mk
index 39bac377c3..71307ab822 100644
--- a/packages/lakka/libretro_cores/genesis_plus_gx/package.mk
+++ b/packages/lakka/libretro_cores/genesis_plus_gx/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="genesis_plus_gx"
-PKG_VERSION="7d4ae7da0c9cbfb6dc7cc8caac701ee948db53c5"
+PKG_VERSION="46652c7fd74bd64a99f624b0bd53a768de0ff672"
 PKG_LICENSE="Non-commercial"
 PKG_SITE="https://github.com/ekeeke/Genesis-Plus-GX"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/genesis_plus_gx_wide/package.mk b/packages/lakka/libretro_cores/genesis_plus_gx_wide/package.mk
index abe4fb1be1..6e9d3becaa 100644
--- a/packages/lakka/libretro_cores/genesis_plus_gx_wide/package.mk
+++ b/packages/lakka/libretro_cores/genesis_plus_gx_wide/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="genesis_plus_gx_wide"
-PKG_VERSION="fa71a1640f6a01025a045b616007c2fe4effd384"
+PKG_VERSION="f634cc53fd8dd8c09a24aac5314d147635857e62"
 PKG_LICENSE="Non-commercial"
 PKG_SITE="https://github.com/libretro/Genesis-Plus-GX-Wide"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/gme/package.mk b/packages/lakka/libretro_cores/gme/package.mk
index 8a47b34b82..5a2002a9d3 100644
--- a/packages/lakka/libretro_cores/gme/package.mk
+++ b/packages/lakka/libretro_cores/gme/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="gme"
-PKG_VERSION="5fb30ebb8871624cecc07813212fb643fa8449f4"
+PKG_VERSION="635b1e9124195eeb44d8a865f258cc2c92846694"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/libretro-gme"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/gpsp/package.mk b/packages/lakka/libretro_cores/gpsp/package.mk
index 5d8162e541..8684cfe6b6 100644
--- a/packages/lakka/libretro_cores/gpsp/package.mk
+++ b/packages/lakka/libretro_cores/gpsp/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="gpsp"
-PKG_VERSION="f0f0b31f9ab95946965b75fed8d31e19290f3d43"
+PKG_VERSION="81649a2c8075201bb823cce8fdf16a31c92a3b6c"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/gpsp"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/gw_libretro/package.mk b/packages/lakka/libretro_cores/gw_libretro/package.mk
index ca03fd51c4..915c4ee371 100644
--- a/packages/lakka/libretro_cores/gw_libretro/package.mk
+++ b/packages/lakka/libretro_cores/gw_libretro/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="gw_libretro"
-PKG_VERSION="d08a08154ce8ed8e9de80582c108f157e4c6b226"
+PKG_VERSION="19a1cb3105ca4a82139fb4994e7995fd956f6f8d"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/gw-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/handy/package.mk b/packages/lakka/libretro_cores/handy/package.mk
index 527dac5a51..245802c4eb 100644
--- a/packages/lakka/libretro_cores/handy/package.mk
+++ b/packages/lakka/libretro_cores/handy/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="handy"
-PKG_VERSION="8133b0a08be2ac7da79a1d79255dbf8787c8be1d"
+PKG_VERSION="517bb2d02909271836604c01c8f09a79ad605297"
 PKG_LICENSE="Zlib"
 PKG_SITE="https://github.com/libretro/libretro-handy"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/hatari/package.mk b/packages/lakka/libretro_cores/hatari/package.mk
index 04eeaac3bf..b477dfd7be 100644
--- a/packages/lakka/libretro_cores/hatari/package.mk
+++ b/packages/lakka/libretro_cores/hatari/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="hatari"
-PKG_VERSION="e5e36a5262cfeadc3d1c7b411b7a70719c4f293c"
+PKG_VERSION="1ebf0a0488580ef95c0b28f02223b31813c867c5"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/hatari"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/kronos/package.mk b/packages/lakka/libretro_cores/kronos/package.mk
index 860928e22b..dd114a6b6c 100644
--- a/packages/lakka/libretro_cores/kronos/package.mk
+++ b/packages/lakka/libretro_cores/kronos/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="kronos"
-PKG_VERSION="a387dec04313a7491415e4e150e5613f4a4a3e6c"
+PKG_VERSION="869c46c1d3684d820f5a368bcae1cc95fe5a9888"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/yabause"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/lutro/package.mk b/packages/lakka/libretro_cores/lutro/package.mk
index eddb085d86..f80bf00dfd 100644
--- a/packages/lakka/libretro_cores/lutro/package.mk
+++ b/packages/lakka/libretro_cores/lutro/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="lutro"
-PKG_VERSION="cc6b06db98bcb7d0379082a876fed870e203ccb3"
+PKG_VERSION="bb24fcb89158ca1c72865af4735ca744b774ab64"
 PKG_LICENSE="MIT"
 PKG_SITE="https://github.com/libretro/libretro-lutro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/mame/package.mk b/packages/lakka/libretro_cores/mame/package.mk
index 534991acbb..f2c274ca9e 100644
--- a/packages/lakka/libretro_cores/mame/package.mk
+++ b/packages/lakka/libretro_cores/mame/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="mame"
-PKG_VERSION="b7dd999590717638ceade2e24d16d63147aa12ad"
+PKG_VERSION="8db2ff91a87e3c3e7b7fa2bd44beb5e5f7a1857d"
 PKG_LICENSE="MAME"
 PKG_SITE="https://github.com/libretro/mame"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/mame2003_plus/package.mk b/packages/lakka/libretro_cores/mame2003_plus/package.mk
index 5692801a04..256ff66bfb 100644
--- a/packages/lakka/libretro_cores/mame2003_plus/package.mk
+++ b/packages/lakka/libretro_cores/mame2003_plus/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="mame2003_plus"
-PKG_VERSION="e5ee29ecb8182952f861f22516e5791625fe2671"
+PKG_VERSION="00302ae29c53c42ffb252da8244cecdac3d5b2d9"
 PKG_LICENSE="MAME"
 PKG_SITE="https://github.com/libretro/mame2003-plus-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/mame2010/package.mk b/packages/lakka/libretro_cores/mame2010/package.mk
index 34a21c6307..c9d91ea0d1 100644
--- a/packages/lakka/libretro_cores/mame2010/package.mk
+++ b/packages/lakka/libretro_cores/mame2010/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="mame2010"
-PKG_VERSION="932e6f2c4f13b67b29ab33428a4037dee9a236a8"
+PKG_VERSION="5f524dd5fca63ec1dcf5cca63885286109937587"
 PKG_LICENSE="MAME"
 PKG_SITE="https://github.com/libretro/mame2010-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/mame2015/package.mk b/packages/lakka/libretro_cores/mame2015/package.mk
index 4e873b0b86..38c0291917 100644
--- a/packages/lakka/libretro_cores/mame2015/package.mk
+++ b/packages/lakka/libretro_cores/mame2015/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="mame2015"
-PKG_VERSION="e6a7aa4d53726e61498f68d6b8e2c092a2169fa2"
+PKG_VERSION="2599c8aeaf84f62fe16ea00daa460a19298c121c"
 PKG_ARCH="x86_64 aarch64 arm"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/mame2015-libretro"
diff --git a/packages/lakka/libretro_cores/melonds/package.mk b/packages/lakka/libretro_cores/melonds/package.mk
index 2ea65da153..800ab224bd 100644
--- a/packages/lakka/libretro_cores/melonds/package.mk
+++ b/packages/lakka/libretro_cores/melonds/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="melonds"
-PKG_VERSION="e93ec3e462d3dfc1556781510a3cee113f02abb2"
+PKG_VERSION="1eebc828769f2619bd79f39545573a991391de5d"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/melonds"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/mesen/package.mk b/packages/lakka/libretro_cores/mesen/package.mk
index 50a97fdd15..6b90e09793 100644
--- a/packages/lakka/libretro_cores/mesen/package.mk
+++ b/packages/lakka/libretro_cores/mesen/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="mesen"
-PKG_VERSION="bb9ea02eba28682986044a6f49329ec533aa26ba"
+PKG_VERSION="9b412c1533a6d7eec7b2904775cbd26c21f02119"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/Mesen"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/mesen_s/package.mk b/packages/lakka/libretro_cores/mesen_s/package.mk
index 760afe9a81..ddd37889eb 100644
--- a/packages/lakka/libretro_cores/mesen_s/package.mk
+++ b/packages/lakka/libretro_cores/mesen_s/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="mesen_s"
-PKG_VERSION="b0b53409eecb696fb13f411ffde72e8f576feb09"
+PKG_VERSION="32a7adfb4edb029324253cb3632dfc6599ad1aa8"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/Mesen-S"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/mgba/package.mk b/packages/lakka/libretro_cores/mgba/package.mk
index d44c87e377..d5fa5bdead 100644
--- a/packages/lakka/libretro_cores/mgba/package.mk
+++ b/packages/lakka/libretro_cores/mgba/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="mgba"
-PKG_VERSION="5d48e0744059ebf38a4e937b256ffd5df4e0d103"
+PKG_VERSION="ec5ecb26deba8d7ac830fc66ade9fac0eeaeb4ae"
 PKG_LICENSE="MPLv2.0"
 PKG_SITE="https://github.com/libretro/mgba"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/mrboom/package.mk b/packages/lakka/libretro_cores/mrboom/package.mk
index b4412baffc..e4337d7b69 100644
--- a/packages/lakka/libretro_cores/mrboom/package.mk
+++ b/packages/lakka/libretro_cores/mrboom/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="mrboom"
-PKG_VERSION="64e874abdd11b845c013ea74e5a62ed1e12d09e2"
+PKG_VERSION="1bc0933b71051411404cdc092b14ade17efb2027"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/mrboom-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/mu/package.mk b/packages/lakka/libretro_cores/mu/package.mk
index bf9699db2e..244d290c31 100644
--- a/packages/lakka/libretro_cores/mu/package.mk
+++ b/packages/lakka/libretro_cores/mu/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="mu"
-PKG_VERSION="1361da04014d77b8bec13e050a71bf7bf8ce5543"
+PKG_VERSION="4a3fee96e7ccc3efe410d2fc828f0cb4aa968747"
 PKG_LICENSE="Non-commercial"
 PKG_SITE="https://github.com/libretro/Mu"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/mupen64plus_next/package.mk b/packages/lakka/libretro_cores/mupen64plus_next/package.mk
index de3d772d29..9ac29fda5a 100644
--- a/packages/lakka/libretro_cores/mupen64plus_next/package.mk
+++ b/packages/lakka/libretro_cores/mupen64plus_next/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="mupen64plus_next"
-PKG_VERSION="4684cfa56ae7752be284eaaa165c1dc34ec63eb7"
+PKG_VERSION="1b67122ff6a923c93a56ff94273e3768a6da5dff"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/mupen64plus-libretro-nx"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/neocd/package.mk b/packages/lakka/libretro_cores/neocd/package.mk
index 44a9ff4475..66daf29f1c 100644
--- a/packages/lakka/libretro_cores/neocd/package.mk
+++ b/packages/lakka/libretro_cores/neocd/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="neocd"
-PKG_VERSION="327aeceecdf71c8a0c0af3d6dc53686c94fe44ad"
+PKG_VERSION="53f5453311a1ac43700fedb2317c810586f9ccf5"
 PKG_LICENSE="LGPLv3"
 PKG_SITE="https://github.com/libretro/neocd_libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/nestopia/package.mk b/packages/lakka/libretro_cores/nestopia/package.mk
index 488d41bd46..178c5c5dac 100644
--- a/packages/lakka/libretro_cores/nestopia/package.mk
+++ b/packages/lakka/libretro_cores/nestopia/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="nestopia"
-PKG_VERSION="a9e197f2583ef4f36e9e77d930a677e63a2c2f62"
+PKG_VERSION="5c360e55d5437ecd3520568ee44cf1af63d4696a"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/nestopia"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/np2kai/package.mk b/packages/lakka/libretro_cores/np2kai/package.mk
index 689302f530..cbfc2a9c42 100644
--- a/packages/lakka/libretro_cores/np2kai/package.mk
+++ b/packages/lakka/libretro_cores/np2kai/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="np2kai"
-PKG_VERSION="2b09ea6a589cdcae27bca27160b3f82638fbb45d"
+PKG_VERSION="606fafa7081b62df5f4727c34560da23927c21cd"
 PKG_LICENSE="MIT"
 PKG_SITE="https://github.com/AZO234/NP2kai"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/nxengine/package.mk b/packages/lakka/libretro_cores/nxengine/package.mk
index 86dfa7b89f..b6853135fb 100644
--- a/packages/lakka/libretro_cores/nxengine/package.mk
+++ b/packages/lakka/libretro_cores/nxengine/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="nxengine"
-PKG_VERSION="fc102fa3eeda1a0133f80474d6671af4cf7612a6"
+PKG_VERSION="e271c6262d73f07e5d92d285503f1c049801c51a"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/nxengine-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/o2em/package.mk b/packages/lakka/libretro_cores/o2em/package.mk
index 19251d6b76..fdb701e2c8 100644
--- a/packages/lakka/libretro_cores/o2em/package.mk
+++ b/packages/lakka/libretro_cores/o2em/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="o2em"
-PKG_VERSION="641f06d67d192a0677ec861fcb731d3ce8da0f87"
+PKG_VERSION="3303cc15e4323280084471f694f6d34c78199725"
 PKG_LICENSE="Artistic License"
 PKG_SITE="https://github.com/libretro/libretro-o2em"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/openlara/package.mk b/packages/lakka/libretro_cores/openlara/package.mk
index 1536bfcfe1..d61797f56b 100644
--- a/packages/lakka/libretro_cores/openlara/package.mk
+++ b/packages/lakka/libretro_cores/openlara/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="openlara"
-PKG_VERSION="fe88d74673344afc71891717b2474d33f90c165f"
+PKG_VERSION="96989ac41ae55a42b19916dc8191f74be40e1b07"
 PKG_LICENSE="BSD"
 PKG_SITE="https://github.com/libretro/openlara"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/openlara/patches/openlara-002-mesa-fix.patch b/packages/lakka/libretro_cores/openlara/patches/openlara-002-mesa-fix.patch
index bac28b8761..1dc8812567 100644
--- a/packages/lakka/libretro_cores/openlara/patches/openlara-002-mesa-fix.patch
+++ b/packages/lakka/libretro_cores/openlara/patches/openlara-002-mesa-fix.patch
@@ -1,5 +1,3 @@
-Zhang Ning <832666+zhangn1985@users.noreply.github.com>
-https://github.com/libretro/OpenLara/issues/32
 diff --git a/src/platform/libretro/libretro-common/glsym/glgen.py b/src/platform/libretro/libretro-common/glsym/glgen.py
 index a599e55..270e2c2 100644
 --- a/src/platform/libretro/libretro-common/glsym/glgen.py
@@ -23,14 +21,15 @@ index a599e55..270e2c2 100644
  def dump(f, lines):
     f.write('\n'.join(lines))
 diff --git a/src/platform/libretro/main.cpp b/src/platform/libretro/main.cpp
-index abccf05..c9546d1 100644
+index 38a752a..545b954 100644
 --- a/src/platform/libretro/main.cpp
 +++ b/src/platform/libretro/main.cpp
-@@ -4,7 +4,7 @@
- #include <string.h>
- #include <math.h>
- 
+@@ -7,7 +7,7 @@
+ #ifdef _WIN32
+ #include "glsym.h" // Use local modified glsym.h to allow compilation on 64bit Windows
+ #else
 -#include <glsym/glsym.h>
 +#include <glsym/rglgen.h>
+ #endif
+
  #include <libretro.h>
- #include <file/file_path.h>
diff --git a/packages/lakka/libretro_cores/opera/package.mk b/packages/lakka/libretro_cores/opera/package.mk
index a7f9f50a02..99d1834f86 100644
--- a/packages/lakka/libretro_cores/opera/package.mk
+++ b/packages/lakka/libretro_cores/opera/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="opera"
-PKG_VERSION="3849c969c64b82e622a7655b327fa94bc5a4c7cc"
+PKG_VERSION="8a49bb8877611037438aeb857cb182f41ee0e3a1"
 PKG_LICENSE="LGPL with additional notes"
 PKG_SITE="https://github.com/libretro/opera-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/parallel_n64/package.mk b/packages/lakka/libretro_cores/parallel_n64/package.mk
index 2b3954a4a4..0967cbb599 100644
--- a/packages/lakka/libretro_cores/parallel_n64/package.mk
+++ b/packages/lakka/libretro_cores/parallel_n64/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="parallel_n64"
-PKG_VERSION="b804ab1a199d6ff1f8fef4aa7dfcf663990e430b"
+PKG_VERSION="a03fdcba6b2e9993f050b50112f597ce2f44fa2c"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/parallel-n64"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/pcsx2/package.mk b/packages/lakka/libretro_cores/pcsx2/package.mk
index 9101675e17..8e9c980563 100644
--- a/packages/lakka/libretro_cores/pcsx2/package.mk
+++ b/packages/lakka/libretro_cores/pcsx2/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="pcsx2"
-PKG_VERSION="0251730a21d7238856d79aa25e2942b48edb38f6"
+PKG_VERSION="4c0613344e2affb90ea0d9552a1d1bfd5c654944"
 PKG_ARCH="x86_64"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/pcsx2"
diff --git a/packages/lakka/libretro_cores/pcsx_rearmed/package.mk b/packages/lakka/libretro_cores/pcsx_rearmed/package.mk
index 6a1d57cbc4..25ed205a95 100644
--- a/packages/lakka/libretro_cores/pcsx_rearmed/package.mk
+++ b/packages/lakka/libretro_cores/pcsx_rearmed/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="pcsx_rearmed"
-PKG_VERSION="e24732050e902bd5402b2b7da7c391d2ca8fa799"
+PKG_VERSION="a4e249a1373cf6269e1e4e0d60105e72210e67d3"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/pcsx_rearmed"
 PKG_URL="${PKG_SITE}.git"
@@ -26,9 +26,9 @@ fi
 if [ "${ARCH}" = "arm" ]; then
   PKG_MAKE_OPTS_TARGET+=" DYNAREC=ari64"
   if target_has_feature neon ; then
-    PKG_MAKE_OPTS_TARGET+=" HAVE_NEON=1 BUILTIN_GPU=neon"
+    PKG_MAKE_OPTS_TARGET+=" HAVE_NEON_ASM=1 BUILTIN_GPU=neon"
   else
-    PKG_MAKE_OPTS_TARGET+=" HAVE_NEON=0 BUILTIN_GPU=unai"
+    PKG_MAKE_OPTS_TARGET+=" HAVE_NEON_ASM=0 BUILTIN_GPU=unai"
   fi
   if [ "${DEVICE}" = "OdroidGoAdvance" ]; then
     sed -e "s|armv8-a|armv8-a+crc|" \
diff --git a/packages/lakka/libretro_cores/picodrive/package.mk b/packages/lakka/libretro_cores/picodrive/package.mk
index fbb7304eba..3a26aa8615 100644
--- a/packages/lakka/libretro_cores/picodrive/package.mk
+++ b/packages/lakka/libretro_cores/picodrive/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="picodrive"
-PKG_VERSION="bb6a52fe60e6f3bdcd17effe75e68fd0f8c44ba7"
+PKG_VERSION="0a4ec83cbfaebb65fb1c40f26ffaf28131f9003b"
 PKG_LICENSE="MAME"
 PKG_SITE="https://github.com/libretro/picodrive"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/play/package.mk b/packages/lakka/libretro_cores/play/package.mk
index 35119545dc..b85617b60a 100644
--- a/packages/lakka/libretro_cores/play/package.mk
+++ b/packages/lakka/libretro_cores/play/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="play"
-PKG_VERSION="fd530d0d53c13e73601dea23637b86bff0d950ce"
+PKG_VERSION="3b1b3d7fcf3b9fef7c89de4a6bf04bcb0d32d176"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/jpd002/Play-"
 PKG_URL="${PKG_SITE}.git"
@@ -12,8 +12,7 @@ PKG_CMAKE_OPTS_TARGET="-DBUILD_LIBRETRO_CORE=yes \
                        -DBUILD_TESTS=no \
                        -DENABLE_AMAZON_S3=no \
                        -DUSE_GLEW=no \
-                       -DCMAKE_BUILD_TYPE=Release \
-                       --target play_libretro"
+                       -DCMAKE_BUILD_TYPE=Release"
 
 if [ "${OPENGL_SUPPORT}" = "yes" ]; then
   PKG_DEPENDS_TARGET+=" ${OPENGL} glu"
diff --git a/packages/lakka/libretro_cores/pocketcdg/package.mk b/packages/lakka/libretro_cores/pocketcdg/package.mk
index 52f863bb22..1d1406b76c 100644
--- a/packages/lakka/libretro_cores/pocketcdg/package.mk
+++ b/packages/lakka/libretro_cores/pocketcdg/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="pocketcdg"
-PKG_VERSION="1559b4e309888690722a86b897582f62dd598810"
+PKG_VERSION="afb88fb20b39e548b8a95791839f59b111892f21"
 PKG_LICENSE="MIT"
 PKG_SITE="https://github.com/libretro/libretro-pocketcdg"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/pokemini/package.mk b/packages/lakka/libretro_cores/pokemini/package.mk
index c349dc4531..1e045245d4 100644
--- a/packages/lakka/libretro_cores/pokemini/package.mk
+++ b/packages/lakka/libretro_cores/pokemini/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="pokemini"
-PKG_VERSION="684e7ea0950f4df48cd1fbf1160e6af3c262c9f0"
+PKG_VERSION="9ba2c2d98bef98794095f3ef50e22f1a3cbc6166"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/pokemini"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/potator/package.mk b/packages/lakka/libretro_cores/potator/package.mk
index f347a04dc0..7a5855d904 100644
--- a/packages/lakka/libretro_cores/potator/package.mk
+++ b/packages/lakka/libretro_cores/potator/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="potator"
-PKG_VERSION="3e3bfd787306d5a074a91bfd9baee2f9cc3b82f3"
+PKG_VERSION="06ec3b724b6fccda6805e3baa76c00c3bc024f23"
 PKG_LICENSE="GPL3"
 PKG_SITE="https://github.com/libretro/potator"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/ppsspp/package.mk b/packages/lakka/libretro_cores/ppsspp/package.mk
index cceae0139e..4ffa1d9a23 100644
--- a/packages/lakka/libretro_cores/ppsspp/package.mk
+++ b/packages/lakka/libretro_cores/ppsspp/package.mk
@@ -1,20 +1,24 @@
 PKG_NAME="ppsspp"
-PKG_VERSION="ce0a45cf0fcdd5bebf32208b9998f68dfc1107b7"
-PKG_LICENSE="GPLv2"
+PKG_VERSION="9fe6338e3bf397f8a009a51a282c139dfa180eb6" #v1.13.2
+PKG_LICENSE="GPL-2.0-or-later"
 PKG_SITE="https://github.com/hrydgard/ppsspp"
-PKG_URL="${PKG_SITE}.git"
-PKG_DEPENDS_TARGET="toolchain libzip libpng"
-PKG_LONGDESC="Libretro port of PPSSPP"
-PKG_TOOLCHAIN="cmake"
-PKG_LR_UPDATE_TAG="yes"
+PKG_URL="https://github.com/hrydgard/ppsspp.git"
+PKG_DEPENDS_TARGET="toolchain linux glibc libzip libpng zstd zlib ffmpeg bzip2 openssl speex"
+PKG_LONGDESC="A PSP emulator for Android, Windows, Mac, Linux and Blackberry 10, written in C++."
+GET_HANDLER_SUPPORT="git"
+PKG_GIT_CLONE_BRANCH="master"
+PKG_GIT_CLONE_SINGLE="yes"
+PKG_BUILD_FLAGS="-sysroot"
 
 PKG_CMAKE_OPTS_TARGET="-DLIBRETRO=ON \
                        -DCMAKE_BUILD_TYPE=Release \
                        -DUSE_FFMPEG=ON \
-                       -DUSE_SYSTEM_FFMPEG=OFF \
+                       -DUSE_SYSTEM_FFMPEG=ON \
+                       -DUSE_SYSTEM_LIBZIP=ON \
+                       -DUSE_SYSTEM_LIBPNG=ON \
+                       -DUSE_SYSTEM_ZSTD=ON \
                        -DUSE_DISCORD=OFF \
-                       -DUSE_MINIUPNPC=OFF \
-                       --target ppsspp_libretro"
+                       -DUSE_MINIUPNPC=OFF"
 
 if [ "${OPENGL_SUPPORT}" = "yes" ]; then
   PKG_DEPENDS_TARGET+=" ${OPENGL}"
diff --git a/packages/lakka/libretro_cores/ppsspp/patches/ppsspp-fix_strcasecmp.patch b/packages/lakka/libretro_cores/ppsspp/patches/ppsspp-fix_strcasecmp.patch
deleted file mode 100644
index 509b781535..0000000000
--- a/packages/lakka/libretro_cores/ppsspp/patches/ppsspp-fix_strcasecmp.patch
+++ /dev/null
@@ -1,13 +0,0 @@
-diff --git a/ext/libzip/compat.h b/ext/libzip/compat.h
-index a5cf72804..b41a53ba5 100644
---- a/ext/libzip/compat.h
-+++ b/ext/libzip/compat.h
-@@ -38,6 +38,8 @@
- 
- #include "config.h"
- 
-+#include <strings.h>
-+
- /* to have *_MAX definitions for all types when compiling with g++ */
- #define __STDC_LIMIT_MACROS
- 
diff --git a/packages/lakka/libretro_cores/ppsspp/patches/ppsspp-ignore-sdl.patch b/packages/lakka/libretro_cores/ppsspp/patches/ppsspp-ignore-sdl.patch
deleted file mode 100644
index c93f9dffcd..0000000000
--- a/packages/lakka/libretro_cores/ppsspp/patches/ppsspp-ignore-sdl.patch
+++ /dev/null
@@ -1,13 +0,0 @@
-diff --git a/CMakeLists.txt b/CMakeLists.txt
-index 0a5fceb50..9192de994 100644
---- a/CMakeLists.txt
-+++ b/CMakeLists.txt
-@@ -997,7 +997,7 @@ elseif(WIN32)
- 	else()
- 		link_directories(dx9sdk/Lib/x86)
- 	endif()
--elseif(TARGET SDL2::SDL2)
-+elseif(TARGET SDL2::SDL2 AND NOT LIBRETRO)
- 	set(TargetBin PPSSPPSDL)
- 	# Require SDL
- 	add_definitions(-DSDL)
diff --git a/packages/lakka/libretro_cores/prboom/package.mk b/packages/lakka/libretro_cores/prboom/package.mk
index 69a78a051a..8072a621d3 100644
--- a/packages/lakka/libretro_cores/prboom/package.mk
+++ b/packages/lakka/libretro_cores/prboom/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="prboom"
-PKG_VERSION="b22a6b19fd976a14374db9083baea9c91b079106"
+PKG_VERSION="4e671fa0a4b7b892e17ac4e1803c9d627653a4c1"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/libretro-prboom"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/prosystem/package.mk b/packages/lakka/libretro_cores/prosystem/package.mk
index 978342951d..9782602a13 100644
--- a/packages/lakka/libretro_cores/prosystem/package.mk
+++ b/packages/lakka/libretro_cores/prosystem/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="prosystem"
-PKG_VERSION="fbf62c3dacaac694f7ec26cf9be10a51b27271e7"
+PKG_VERSION="cf544d3c8e40ff197ea5bb177a1269db31077803"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/prosystem-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/puae/package.mk b/packages/lakka/libretro_cores/puae/package.mk
index c6ad7b70c3..b0899ec50c 100644
--- a/packages/lakka/libretro_cores/puae/package.mk
+++ b/packages/lakka/libretro_cores/puae/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="puae"
-PKG_VERSION="fd52745fb0955dd0a4efcceb049f5f7f44d69610"
+PKG_VERSION="d9a8dfbde7f6967fea3cffe09cd87e1d79a1a3fd"
 PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/libretro/libretro-uae"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/puae2021/package.mk b/packages/lakka/libretro_cores/puae2021/package.mk
index 0534eb6376..e939b15e8c 100644
--- a/packages/lakka/libretro_cores/puae2021/package.mk
+++ b/packages/lakka/libretro_cores/puae2021/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="puae2021"
-PKG_VERSION="7a2f6f1da8b4c949c26806a3fa23f7455f44cd39"
+PKG_VERSION="d6576ca8c544c579a9d5de64ed3d38f89c876dbd"
 PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/libretro/libretro-uae"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/px68k/package.mk b/packages/lakka/libretro_cores/px68k/package.mk
index 1731f8b49e..28793eed42 100644
--- a/packages/lakka/libretro_cores/px68k/package.mk
+++ b/packages/lakka/libretro_cores/px68k/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="px68k"
-PKG_VERSION="0183381d655be213d9c27207e1a29ad372a7a16d"
+PKG_VERSION="84a982f966a015bb6e9bae5f3fd756c3a3fa3369"
 PKG_LICENSE="Unknown"
 PKG_SITE="https://github.com/libretro/px68k-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/quicknes/package.mk b/packages/lakka/libretro_cores/quicknes/package.mk
index b660112ce6..3db67b72b1 100644
--- a/packages/lakka/libretro_cores/quicknes/package.mk
+++ b/packages/lakka/libretro_cores/quicknes/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="quicknes"
-PKG_VERSION="e6f08c165af45fc2d2f26c80ba0cfc33e26f9cfe"
+PKG_VERSION="1b88a09f1c386ff9ee46bb371583ae04c5cb5dd0"
 PKG_LICENSE="LGPLv2.1+"
 PKG_SITE="https://github.com/libretro/QuickNES_Core"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/race/package.mk b/packages/lakka/libretro_cores/race/package.mk
index f62188d02f..9a10c6fb14 100644
--- a/packages/lakka/libretro_cores/race/package.mk
+++ b/packages/lakka/libretro_cores/race/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="race"
-PKG_VERSION="b8264bbaa0878e2d0ebae8ca60bd886c5971e255"
+PKG_VERSION="cb002043c65276dac592ed0214ae8a6bba82e2b1"
 PKG_LICENSE="GPL2"
 PKG_SITE="https://github.com/libretro/RACE"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/retro8/package.mk b/packages/lakka/libretro_cores/retro8/package.mk
index 4ce094a396..7e47acd668 100644
--- a/packages/lakka/libretro_cores/retro8/package.mk
+++ b/packages/lakka/libretro_cores/retro8/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="retro8"
-PKG_VERSION="afa143ac123034fb7907f05282ac746c214deb1f"
+PKG_VERSION="bc388ec7d217a08265d116aaa74afc0ca3f204f5"
 PKG_LICENSE="GPL3"
 PKG_SITE="https://github.com/libretro/retro8"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/same_cdi/package.mk b/packages/lakka/libretro_cores/same_cdi/package.mk
index 2b14fa38bd..329ea581ce 100644
--- a/packages/lakka/libretro_cores/same_cdi/package.mk
+++ b/packages/lakka/libretro_cores/same_cdi/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="same_cdi"
-PKG_VERSION="dcc76a5b6d3fb5da9ca29209e1a66bbc1396f83a"
+PKG_VERSION="a68ef22890b519b4adc35c01b5eeafbe50629e98"
 PKG_LICENSE="MAME"
 PKG_SITE="https://github.com/libretro/same_cdi"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/sameboy/package.mk b/packages/lakka/libretro_cores/sameboy/package.mk
index ac1d565837..fb48832d53 100644
--- a/packages/lakka/libretro_cores/sameboy/package.mk
+++ b/packages/lakka/libretro_cores/sameboy/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="sameboy"
-PKG_VERSION="b154b7d3d885a3cf31203f0b8f50d3b37c8b742b"
+PKG_VERSION="09138330990da32362246c7034cf4de2ea0a2a2b"
 PKG_LICENSE="MIT"
 PKG_SITE="https://github.com/libretro/sameboy"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/sameduck/package.mk b/packages/lakka/libretro_cores/sameduck/package.mk
index eae2da3d17..72c0433fcc 100644
--- a/packages/lakka/libretro_cores/sameduck/package.mk
+++ b/packages/lakka/libretro_cores/sameduck/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="sameduck"
-PKG_VERSION="a5e5462f79164d21053f81fc680bfa9f59a2bf59"
+PKG_VERSION="5cc014b23af963f9e9a77091a1e6ff7a7852e8a7"
 PKG_LICENSE="MIT"
 PKG_SITE="https://github.com/LIJI32/SameBoy"
 PKG_GIT_CLONE_BRANCH="SameDuck"
diff --git a/packages/lakka/libretro_cores/snes9x/package.mk b/packages/lakka/libretro_cores/snes9x/package.mk
index 3fb83721a4..03a5755b95 100644
--- a/packages/lakka/libretro_cores/snes9x/package.mk
+++ b/packages/lakka/libretro_cores/snes9x/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="snes9x"
-PKG_VERSION="92e5907e35ee2e17c7d7f3a1d77f66544926898c"
+PKG_VERSION="2ed8d687f199ad826b3d9cb527649c48784e12f8"
 PKG_LICENSE="Non-commercial"
 PKG_SITE="https://github.com/libretro/snes9x"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/snes9x2002/package.mk b/packages/lakka/libretro_cores/snes9x2002/package.mk
index 5809cd1292..fde6c8239d 100644
--- a/packages/lakka/libretro_cores/snes9x2002/package.mk
+++ b/packages/lakka/libretro_cores/snes9x2002/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="snes9x2002"
-PKG_VERSION="c4397de75a5f11403d154abd935e39fe969bca94"
+PKG_VERSION="540baad622d9833bba7e0696193cb06f5f02f564"
 PKG_LICENSE="Non-commercial"
 PKG_SITE="https://github.com/libretro/snes9x2002"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/snes9x2005/package.mk b/packages/lakka/libretro_cores/snes9x2005/package.mk
index 63fa128afd..cee7785f02 100644
--- a/packages/lakka/libretro_cores/snes9x2005/package.mk
+++ b/packages/lakka/libretro_cores/snes9x2005/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="snes9x2005"
-PKG_VERSION="23f759bc4bf2e39733296f7749e446418e3cd0f3"
+PKG_VERSION="fd45b0e055bce6cff3acde77414558784e93e7d0"
 PKG_LICENSE="Non-commercial"
 PKG_SITE="https://github.com/libretro/snes9x2005"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/snes9x2005_plus/package.mk b/packages/lakka/libretro_cores/snes9x2005_plus/package.mk
index c2b3161ef0..9a70dab929 100644
--- a/packages/lakka/libretro_cores/snes9x2005_plus/package.mk
+++ b/packages/lakka/libretro_cores/snes9x2005_plus/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="snes9x2005_plus"
-PKG_VERSION="23f759bc4bf2e39733296f7749e446418e3cd0f3"
+PKG_VERSION="fd45b0e055bce6cff3acde77414558784e93e7d0"
 PKG_LICENSE="Non-commercial"
 PKG_SITE="https://github.com/libretro/snes9x2005"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/snes9x2010/package.mk b/packages/lakka/libretro_cores/snes9x2010/package.mk
index 46f1ac3682..68cb11c1da 100644
--- a/packages/lakka/libretro_cores/snes9x2010/package.mk
+++ b/packages/lakka/libretro_cores/snes9x2010/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="snes9x2010"
-PKG_VERSION="c98224bc74aa0bbf355d128b22e4a2a4e94215b0"
+PKG_VERSION="e86e54624a7910a64a9a744e3734d4067c48d240"
 PKG_LICENSE="Non-commercial"
 PKG_SITE="https://github.com/libretro/snes9x2010"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/stella/package.mk b/packages/lakka/libretro_cores/stella/package.mk
index 403443968d..c3dbcacec5 100644
--- a/packages/lakka/libretro_cores/stella/package.mk
+++ b/packages/lakka/libretro_cores/stella/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="stella"
-PKG_VERSION="df4900828c60d9df26827b0a6c969b213ac6250f"
+PKG_VERSION="dcc3eebab194aff86015e1b8c7fc8067de8cf34d"
 PKG_LICENSE="GPL2"
 PKG_SITE="https://github.com/stella-emu/stella"
 PKG_URL="${PKG_SITE}.git"
@@ -7,10 +7,10 @@ PKG_DEPENDS_TARGET="toolchain"
 PKG_LONGDESC="Stella is a multi-platform Atari 2600 VCS emulator released under the GNU General Public License (GPL)."
 PKG_TOOLCHAIN="make"
 
-PKG_MAKE_OPTS_TARGET="-C ../src/libretro -f Makefile"
+PKG_MAKE_OPTS_TARGET="-C ../src/os/libretro -f Makefile"
 
 makeinstall_target() {
   mkdir -p ${INSTALL}/usr/lib/libretro
-    cp -v ../src/libretro/stella_libretro.so ${INSTALL}/usr/lib/libretro/
+    cp -v ../src/os/libretro/stella_libretro.so ${INSTALL}/usr/lib/libretro/
 }
 
diff --git a/packages/lakka/libretro_cores/stella/patches/stella-01-do_not-use_static_libgcc_libstdc++.patch b/packages/lakka/libretro_cores/stella/patches/stella-01-do_not-use_static_libgcc_libstdc++.patch
index 36c44c5c43..0195944862 100644
--- a/packages/lakka/libretro_cores/stella/patches/stella-01-do_not-use_static_libgcc_libstdc++.patch
+++ b/packages/lakka/libretro_cores/stella/patches/stella-01-do_not-use_static_libgcc_libstdc++.patch
@@ -1,12 +1,8 @@
-diff --git a/src/libretro/Makefile b/src/libretro/Makefile
-index a170992343..6a828521e5 100644
---- a/src/libretro/Makefile
-+++ b/src/libretro/Makefile
-@@ -57,11 +57,10 @@ ifneq ($(GIT_VERSION)," unknown")
-    CXXFLAGS += -DGIT_VERSION=\"$(GIT_VERSION)\"
- endif
- 
--GET_STATIC_ARG=$(if $(filter $(shell $(CXX) -print-file-name=$1.a),$1.a),,-static-$1)
+diff --git a/src/os/libretro/Makefile b/src/os/libretro/Makefile
+index d8695faef..39fd04013 100644
+--- a/src/os/libretro/Makefile
++++ b/src/os/libretro/Makefile
+@@ -61,7 +61,7 @@ GET_STATIC_ARG=$(if $(filter $(shell $(CXX) -print-file-name=$1.a),$1.a),,-stati
  # Unix
  ifneq (,$(findstring unix,$(platform)))
     CXXFLAGS += $(LTO)
diff --git a/packages/lakka/libretro_cores/swanstation/package.mk b/packages/lakka/libretro_cores/swanstation/package.mk
index d3908a8a5f..4dcb525c7b 100644
--- a/packages/lakka/libretro_cores/swanstation/package.mk
+++ b/packages/lakka/libretro_cores/swanstation/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="swanstation"
-PKG_VERSION="0e27584155a9ecbd63111afb9c8646148aaba46b"
+PKG_VERSION="f2e335bfd4751410dfb24d933f762b9a4fd7fdeb"
 PKG_GIT_CLONE_BRANCH="main"
 PKG_LICENSE="GPL-3.0-or-later"
 PKG_SITE="https://github.com/libretro/swanstation"
diff --git a/packages/lakka/libretro_cores/tgbdual/package.mk b/packages/lakka/libretro_cores/tgbdual/package.mk
index 4b7b34f7b5..ada34e882e 100644
--- a/packages/lakka/libretro_cores/tgbdual/package.mk
+++ b/packages/lakka/libretro_cores/tgbdual/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="tgbdual"
-PKG_VERSION="1e0c4f931d8c5e859e6d3255d67247d7a2987434"
+PKG_VERSION="a6f3018e6a23030afc1873845ee54d4b2d8ec9d3"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/tgbdual-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/theodore/package.mk b/packages/lakka/libretro_cores/theodore/package.mk
index a39c7c4fbb..09b1bf428f 100644
--- a/packages/lakka/libretro_cores/theodore/package.mk
+++ b/packages/lakka/libretro_cores/theodore/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="theodore"
-PKG_VERSION="563fb989f12c7176824b34bbc726bf24d933e86c"
+PKG_VERSION="7889613edede5ba89de1dfe7c05cf8397cf178ba"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/Zlika/theodore"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/tic80/package.mk b/packages/lakka/libretro_cores/tic80/package.mk
index 90f259e801..77b8a3f04e 100644
--- a/packages/lakka/libretro_cores/tic80/package.mk
+++ b/packages/lakka/libretro_cores/tic80/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="tic80"
-PKG_VERSION="e9f62f85a154796c6baaee8a9f6fd0cfdd447019"
+PKG_VERSION="bd6ce86174fc7c9d7d3a86263acf3a7de1b62c11"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/TIC-80"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/tyrquake/package.mk b/packages/lakka/libretro_cores/tyrquake/package.mk
index 4e5af7fd8a..e917233a40 100644
--- a/packages/lakka/libretro_cores/tyrquake/package.mk
+++ b/packages/lakka/libretro_cores/tyrquake/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="tyrquake"
-PKG_VERSION="c516c1fe4266e5f982703d509cc10ceb12167894"
+PKG_VERSION="89f3e032757b9b56e8d62a528a4a2472e60631aa"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/tyrquake"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/uae4arm/package.mk b/packages/lakka/libretro_cores/uae4arm/package.mk
index 79e9fed096..9f63c5130f 100644
--- a/packages/lakka/libretro_cores/uae4arm/package.mk
+++ b/packages/lakka/libretro_cores/uae4arm/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="uae4arm"
-PKG_VERSION="96fd90b21388ae17c4dd83e4208930fdbddf5930"
+PKG_VERSION="177c2f0e892adf2603ada9b150e31beffe0f76c3"
 PKG_ARCH="arm aarch64"
 PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/Chips-fr/uae4arm-rpi"
diff --git a/packages/lakka/libretro_cores/uzem/package.mk b/packages/lakka/libretro_cores/uzem/package.mk
index 7a386d29bb..7a8dab339a 100644
--- a/packages/lakka/libretro_cores/uzem/package.mk
+++ b/packages/lakka/libretro_cores/uzem/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="uzem"
-PKG_VERSION="4c70043e4ad6a8a3aa6326834fba53f2b1c68699"
+PKG_VERSION="08e39e19167727c89fb995e3fa70dde252e6aab0"
 PKG_LICENSE="MIT"
 PKG_SITE="https://github.com/libretro/libretro-uzem"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/vbam/package.mk b/packages/lakka/libretro_cores/vbam/package.mk
index 6f852e4332..38521e9c62 100644
--- a/packages/lakka/libretro_cores/vbam/package.mk
+++ b/packages/lakka/libretro_cores/vbam/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="vbam"
-PKG_VERSION="cbd7d7d2d1b8fab6faf9574b28b72d7a0d05d68b"
+PKG_VERSION="0ca1771a491a331a6c74f19adcd94ede0f6e15db"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/visualboyadvance-m/visualboyadvance-m"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/vecx/package.mk b/packages/lakka/libretro_cores/vecx/package.mk
index 9cd746ba33..be6951a799 100644
--- a/packages/lakka/libretro_cores/vecx/package.mk
+++ b/packages/lakka/libretro_cores/vecx/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="vecx"
-PKG_VERSION="141af284202c86ed0d4ce9030c76954a144287cf"
+PKG_VERSION="b5c17bb7fd4a704f58160bc699322a16d0643396"
 PKG_LICENSE="GPLv2|LGPLv2.1"
 PKG_SITE="https://github.com/libretro/libretro-vecx"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/vice/package.mk b/packages/lakka/libretro_cores/vice/package.mk
index d54a4610cf..ec6ca4a3f5 100644
--- a/packages/lakka/libretro_cores/vice/package.mk
+++ b/packages/lakka/libretro_cores/vice/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="vice"
-PKG_VERSION="af8368a1ab10870486cc35586cf31bb72dfa9a30"
+PKG_VERSION="db8476c7ca5a09e4e6dc9625b4c799a139437fcc"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/vice-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/virtualjaguar/package.mk b/packages/lakka/libretro_cores/virtualjaguar/package.mk
index e0829a9fad..efa9bad83d 100644
--- a/packages/lakka/libretro_cores/virtualjaguar/package.mk
+++ b/packages/lakka/libretro_cores/virtualjaguar/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="virtualjaguar"
-PKG_VERSION="263c979be4ca757c43fb525bd6f0887998e57041"
+PKG_VERSION="2cc06899b839639397b8b30384a191424b6f529d"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/virtualjaguar-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/wasm4/package.mk b/packages/lakka/libretro_cores/wasm4/package.mk
index 3687cf1d9a..c3acabb865 100644
--- a/packages/lakka/libretro_cores/wasm4/package.mk
+++ b/packages/lakka/libretro_cores/wasm4/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="wasm4"
-PKG_VERSION="6e018408335937669732b26da34a67de128f1ea8"
+PKG_VERSION="fb27490a2dbd78a679828f4f11337daaffff2f98"
 PKG_LICENSE="ISC"
 PKG_SITE="https://github.com/aduros/wasm4"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/xmil/package.mk b/packages/lakka/libretro_cores/xmil/package.mk
index 28803f38e1..45777bd6d4 100644
--- a/packages/lakka/libretro_cores/xmil/package.mk
+++ b/packages/lakka/libretro_cores/xmil/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="xmil"
-PKG_VERSION="fc18430bdecae91d470d728a86e6fb3f85d029e6"
+PKG_VERSION="4cb1e4eaab37321904144d1f1a23b2830268e8df"
 PKG_LICENSE="GPL3"
 PKG_SITE="https://github.com/libretro/xmil-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/xrick/package.mk b/packages/lakka/libretro_cores/xrick/package.mk
index c9f6f0e62d..677ed09aca 100644
--- a/packages/lakka/libretro_cores/xrick/package.mk
+++ b/packages/lakka/libretro_cores/xrick/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="xrick"
-PKG_VERSION="10d7318c374de3e0eb056e6c7e823c644a7ce078"
+PKG_VERSION="7ee449e12062ea8d7ca0c945c3a4b63ca0c04268"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/xrick-libretro"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/yabasanshiro/package.mk b/packages/lakka/libretro_cores/yabasanshiro/package.mk
index 8f75c9753f..ca593ef02e 100644
--- a/packages/lakka/libretro_cores/yabasanshiro/package.mk
+++ b/packages/lakka/libretro_cores/yabasanshiro/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="yabasanshiro"
-PKG_VERSION="2848d5053fef1a69f68c600b65a1b9e0d915056c"
+PKG_VERSION="fd459968aae4a251d839174404a346b1f428912a"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/yabause"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/libretro_cores/yabause/package.mk b/packages/lakka/libretro_cores/yabause/package.mk
index a35cbd84db..1607542544 100644
--- a/packages/lakka/libretro_cores/yabause/package.mk
+++ b/packages/lakka/libretro_cores/yabause/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="yabause"
-PKG_VERSION="17dfcd8de4700341d972993501d3a043925675ce"
+PKG_VERSION="c7e02721eddb3de0ec7ae0d61e9e3afa5f586a62"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/yabause"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/retroarch_base/core_info/package.mk b/packages/lakka/retroarch_base/core_info/package.mk
index 6a57e6f1f8..df65aad9e1 100644
--- a/packages/lakka/retroarch_base/core_info/package.mk
+++ b/packages/lakka/retroarch_base/core_info/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="core_info"
-PKG_VERSION="06dac6318c067a6a40e2ea1e1260e74681ada844"
+PKG_VERSION="4369f03d59eac8d2aa32294e1fca73dba30140d3"
 PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/libretro/libretro-core-info"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/retroarch_base/glsl_shaders/package.mk b/packages/lakka/retroarch_base/glsl_shaders/package.mk
index 7567c7a9a6..f0919ad4a1 100644
--- a/packages/lakka/retroarch_base/glsl_shaders/package.mk
+++ b/packages/lakka/retroarch_base/glsl_shaders/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="glsl_shaders"
-PKG_VERSION="8f6b2649489760caea8f36ec6099998854059569"
+PKG_VERSION="66cd3917c3cde8c96b5492bca00c95e3e17b44b9"
 PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/libretro/glsl-shaders"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/retroarch_base/libretro_database/package.mk b/packages/lakka/retroarch_base/libretro_database/package.mk
index 3c511a6031..2e1b3d7259 100644
--- a/packages/lakka/retroarch_base/libretro_database/package.mk
+++ b/packages/lakka/retroarch_base/libretro_database/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="libretro_database"
-PKG_VERSION="f3b10ec262fcbd78da3473f46d244270d12a327e"
+PKG_VERSION="8722b435830c8bf03b0430cc912842140d607b39"
 PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/libretro/libretro-database"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/retroarch_base/retroarch/package.mk b/packages/lakka/retroarch_base/retroarch/package.mk
index bccb5787ba..72d989052e 100644
--- a/packages/lakka/retroarch_base/retroarch/package.mk
+++ b/packages/lakka/retroarch_base/retroarch/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="retroarch"
-PKG_VERSION="9b282aa742b6c3d2f2925ae5a12e2cd7c6b6ad38"
+PKG_VERSION="0bf818af8dda4ef96993889d3ea23f2a2b054136"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/RetroArch"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/retroarch_base/retroarch_assets/package.mk b/packages/lakka/retroarch_base/retroarch_assets/package.mk
index 1b4430ed06..62a6470b49 100644
--- a/packages/lakka/retroarch_base/retroarch_assets/package.mk
+++ b/packages/lakka/retroarch_base/retroarch_assets/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="retroarch_assets"
-PKG_VERSION="f45dd1351e6bfb9f01e12bee8d358ad22a2bb8f9"
+PKG_VERSION="4ec80faf1b5439d1654f407805bb66141b880826"
 PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/libretro/retroarch-assets"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/retroarch_base/retroarch_joypad_autoconfig/package.mk b/packages/lakka/retroarch_base/retroarch_joypad_autoconfig/package.mk
index 259435e599..417cd2f205 100644
--- a/packages/lakka/retroarch_base/retroarch_joypad_autoconfig/package.mk
+++ b/packages/lakka/retroarch_base/retroarch_joypad_autoconfig/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="retroarch_joypad_autoconfig"
-PKG_VERSION="760b057f1865a827762a54a8a32afda2cdd9e170"
+PKG_VERSION="dc625d5623a0e5c6999f372c99548676f9851691"
 PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/libretro/retroarch-joypad-autoconfig"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/retroarch_base/retroarch_overlays/package.mk b/packages/lakka/retroarch_base/retroarch_overlays/package.mk
index 4b60cf2668..db89c49175 100644
--- a/packages/lakka/retroarch_base/retroarch_overlays/package.mk
+++ b/packages/lakka/retroarch_base/retroarch_overlays/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="retroarch_overlays"
-PKG_VERSION="db9744f4e58a740f0f10b04b62af347cd6f01928"
+PKG_VERSION="b3827a2c63834ed9e1902acc7cf9019f64771ed3"
 PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/libretro/common-overlays"
 PKG_URL="${PKG_SITE}.git"
diff --git a/packages/lakka/retroarch_base/slang_shaders/package.mk b/packages/lakka/retroarch_base/slang_shaders/package.mk
index 4729680935..e26d07d648 100644
--- a/packages/lakka/retroarch_base/slang_shaders/package.mk
+++ b/packages/lakka/retroarch_base/slang_shaders/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="slang_shaders"
-PKG_VERSION="148c57d9747f5db124135501ac280b5ca129a83f"
+PKG_VERSION="e5fec95dfe9bee4c753427e657e61ead71e63fb9"
 PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/libretro/slang-shaders"
 PKG_URL="${PKG_SITE}.git"
