diff --git a/packages/devel/liblcf/package.mk b/packages/devel/liblcf/package.mk
index 0cad1583d0..f64991946d 100644
--- a/packages/devel/liblcf/package.mk
+++ b/packages/devel/liblcf/package.mk
@@ -17,12 +17,12 @@
 ################################################################################
 
 PKG_NAME="liblcf"
-PKG_VERSION="01b73de"
+PKG_VERSION="bb9f9e2"
 PKG_ARCH="any"
 PKG_LICENSE="MIT"
 PKG_SITE="https://github.com/EasyRPG/liblcf"
 PKG_URL="$PKG_SITE/archive/$PKG_VERSION.tar.gz"
-PKG_DEPENDS_TARGET="toolchain expat icu69"
+PKG_DEPENDS_TARGET="toolchain expat icu"
 PKG_SECTION="devel"
 PKG_SHORTDESC="Library to handle RPG Maker 2000/2003 and EasyRPG projects"
 
diff --git a/packages/devel/liblcf/patches/liblcf-01-remove-benchmark.patch b/packages/devel/liblcf/patches/liblcf-01-remove-benchmark.patch
new file mode 100644
index 0000000000..eaf7dbdfa3
--- /dev/null
+++ b/packages/devel/liblcf/patches/liblcf-01-remove-benchmark.patch
@@ -0,0 +1,16 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 8a1c885..a27de5b 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -352,11 +352,3 @@ if(NOT TARGET check)
+ endif()
+ add_dependencies(check_lcf test_runner_lcf)
+ add_dependencies(check check_lcf)
+-
+-# benchmarks
+-file(GLOB BENCH_FILES ${CMAKE_CURRENT_SOURCE_DIR}/bench/*.cpp)
+-foreach(i ${BENCH_FILES})
+-	get_filename_component(name "${i}" NAME_WE)
+-	add_executable(bench_${name} ${i})
+-	target_link_libraries(bench_${name} lcf)
+-endforeach()
diff --git a/packages/devel/liblcf/patches/liblcf-02-add-icu-compile-defs.patch b/packages/devel/liblcf/patches/liblcf-02-add-icu-compile-defs.patch
new file mode 100644
index 0000000000..ee90b3cd21
--- /dev/null
+++ b/packages/devel/liblcf/patches/liblcf-02-add-icu-compile-defs.patch
@@ -0,0 +1,13 @@
+--- a/CMakeLists.txt	2020-12-20 15:19:33.535997406 +0100
++++ b/CMakeLists.txt	2020-12-20 15:21:56.337546477 +0100
+@@ -259,7 +259,9 @@
+ else()
+ 	find_package(ICU COMPONENTS i18n uc data REQUIRED)
+ 	target_link_libraries(lcf ICU::i18n ICU::uc ICU::data)
+-	target_compile_definitions(lcf PRIVATE LCF_SUPPORT_ICU=1)
++	INCLUDE(FindPkgConfig)
++	pkg_get_variable(ICUDEFS icu-i18n DEFS)
++	target_compile_definitions(lcf PRIVATE LCF_SUPPORT_ICU=1 ${ICUDEFS})
+ 	list(APPEND LIBLCF_DEPS "icu-i18n")
+ endif()
+ 
diff --git a/packages/lakka/icu69/package.mk b/packages/lakka/icu69/package.mk
deleted file mode 100644
index 2cbeda7442..0000000000
--- a/packages/lakka/icu69/package.mk
+++ /dev/null
@@ -1,30 +0,0 @@
-# SPDX-License-Identifier: GPL-2.0
-# Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
-
-PKG_NAME="icu69"
-PKG_VERSION="69.1"
-PKG_SHA256="39ce83dd5d15c7539dde261733e106a391923f82caf1ce52ecaebb72d93b4579"
-PKG_LICENSE="Custom"
-PKG_SITE="http://www.icu-project.org"
-PKG_URL="https://github.com/unicode-org/icu/archive/release-${PKG_VERSION//./-}.tar.gz"
-PKG_DEPENDS_HOST="toolchain:host"
-PKG_DEPENDS_TARGET="toolchain icu69:host"
-PKG_LONGDESC="International Components for Unicode library."
-PKG_TOOLCHAIN="configure"
-
-PKG_BUILD_FLAGS="-sysroot"
-
-if [ "${DISTRO}" = "Lakka" ]; then
-  PKG_BUILD_FLAGS=""
-fi
-
-configure_package() {
-  PKG_CONFIGURE_SCRIPT="${PKG_BUILD}/icu4c/source/configure"
-  PKG_CONFIGURE_OPTS_TARGET="--disable-layout \
-                             --disable-layoutex \
-                             --enable-renaming \
-                             --disable-samples \
-                             --disable-tests \
-                             --disable-tools \
-                             --with-cross-build=${PKG_BUILD}/.${HOST_NAME}"
-}
diff --git a/packages/lakka/icu69/patches/icu-01_ldflags.patch b/packages/lakka/icu69/patches/icu-01_ldflags.patch
deleted file mode 100644
index 8f740c86cd..0000000000
--- a/packages/lakka/icu69/patches/icu-01_ldflags.patch
+++ /dev/null
@@ -1,12 +0,0 @@
-diff -Naur icu4c-67-1/icu-release-67-1/icu4c/source/config/mh-linux icu4c-67-1-ldflags/icu-release-67-1/icu4c/source/config/mh-linux
---- icu-release-67-1/icu4c/source/config/mh-linux	2020-04-22 19:49:10.000000000 +0200
-+++ icu-release-67-1.ldflags/icu4c/source/config/mh-linux	2020-09-05 17:58:05.635014182 +0200
-@@ -23,7 +23,7 @@
- LD_RPATH_PRE = -Wl,-rpath,
- 
- ## These are the library specific LDFLAGS
--LDFLAGSICUDT=-nodefaultlibs -nostdlib
-+#LDFLAGSICUDT=-nodefaultlibs -nostdlib
- 
- ## Compiler switch to embed a library name
- # The initial tab in the next line is to prevent icu-config from reading it.
diff --git a/packages/libretro/easyrpg/package.mk b/packages/libretro/easyrpg/package.mk
index f9ccf127b3..7ad246b960 100644
--- a/packages/libretro/easyrpg/package.mk
+++ b/packages/libretro/easyrpg/package.mk
@@ -19,19 +19,19 @@
 ################################################################################
 
 PKG_NAME="easyrpg"
-PKG_VERSION="281be71"
+PKG_VERSION="4dd00a6"
 PKG_ARCH="any"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/EasyRPG/Player"
+PKG_GIT_CLONE_BRANCH="0-6-2-stable"
 PKG_URL="$PKG_SITE.git"
-PKG_DEPENDS_TARGET="toolchain zlib libfmt liblcf pixman libspeexdsp mpg123 libsndfile libvorbis opusfile wildmidi libxmp-lite libpng"
+PKG_DEPENDS_TARGET="toolchain zlib libfmt liblcf pixman libspeexdsp mpg123 libsndfile libvorbis opusfile wildmidi libxmp-lite"
 PKG_PRIORITY="optional"
 PKG_SECTION="libretro"
 PKG_SHORTDESC="An unofficial libretro port of the EasyRPG/Player."
 PKG_LONGDESC="An unofficial libretro port of the EasyRPG/Player."
 PKG_TOOLCHAIN="cmake"
 PKG_BUILD_FLAGS="+pic"
-PKG_LR_UPDATE_TAG="yes"
 
 PKG_IS_ADDON="no"
 PKG_AUTORECONF="no"
diff --git a/packages/libretro/easyrpg/patches/cmake_policy.patch b/packages/libretro/easyrpg/patches/cmake_policy.patch
new file mode 100644
index 0000000000..1f5aa306ba
--- /dev/null
+++ b/packages/libretro/easyrpg/patches/cmake_policy.patch
@@ -0,0 +1,10 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index df55873e..e5e1b7cd 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -1,5 +1,4 @@
+ cmake_minimum_required(VERSION 3.7)
+-cmake_policy(SET CMP0091 NEW) # Support MSVC_RUNTIME_LIBRARY
+ 
+ project(EasyRPG_Player VERSION 0.6.2.3 LANGUAGES CXX)
+ 
