diff --git a/packages/lakka/libretro_cores/parallel_n64/package.mk b/packages/lakka/libretro_cores/parallel_n64/package.mk
index 2b3954a4a4..0967cbb599 100644
--- a/packages/lakka/libretro_cores/parallel_n64/package.mk
+++ b/packages/lakka/libretro_cores/parallel_n64/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="parallel_n64"
-PKG_VERSION="b804ab1a199d6ff1f8fef4aa7dfcf663990e430b"
+PKG_VERSION="a03fdcba6b2e9993f050b50112f597ce2f44fa2c"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/parallel-n64"
 PKG_URL="${PKG_SITE}.git"
