diff --git a/packages/devel/glib/package.mk b/packages/devel/glib/package.mk
index c157de8206..f753006a93 100644
--- a/packages/devel/glib/package.mk
+++ b/packages/devel/glib/package.mk
@@ -3,27 +3,23 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="glib"
-PKG_VERSION="2.67.2"
-PKG_SHA256="b41d42d6c572e1e420ffc38a077e0157e0b53b5a94c9647a3dc3701043c3b69b"
+PKG_VERSION="2.76.1"
+PKG_SHA256="43dc0f6a126958f5b454136c4398eab420249c16171a769784486e25f2fda19f"
 PKG_LICENSE="LGPL"
-PKG_SITE="http://www.gtk.org/"
-PKG_URL="http://ftp.gnome.org/pub/gnome/sources/glib/$(get_pkg_version_maj_min)/${PKG_NAME}-${PKG_VERSION}.tar.xz"
-PKG_DEPENDS_HOST="libffi:host Python3:host meson:host ninja:host"
-PKG_DEPENDS_TARGET="toolchain pcre zlib libffi Python3:host util-linux"
+PKG_SITE="https://www.gtk.org/"
+PKG_URL="https://download.gnome.org/sources/glib/$(get_pkg_version_maj_min)/${PKG_NAME}-${PKG_VERSION}.tar.xz"
+PKG_DEPENDS_HOST="libffi:host pcre2:host Python3:host meson:host ninja:host"
+PKG_DEPENDS_TARGET="toolchain pcre2 zlib libffi Python3:host util-linux"
 PKG_LONGDESC="A library which includes support routines for C such as lists, trees, hashes, memory allocation."
-PKG_TOOLCHAIN="meson"
 
 PKG_MESON_OPTS_HOST="-Ddefault_library=static \
-                     -Dinternal_pcre=true \
                      -Dinstalled_tests=false \
                      -Dlibmount=disabled \
                      -Dtests=false"
 
 PKG_MESON_OPTS_TARGET="-Ddefault_library=shared \
-                       -Dinternal_pcre=false \
                        -Dinstalled_tests=false \
                        -Dselinux=disabled \
-                       -Dfam=false \
                        -Dxattr=true \
                        -Dgtk_doc=false \
                        -Dman=false \
@@ -33,12 +29,6 @@ PKG_MESON_OPTS_TARGET="-Ddefault_library=shared \
                        -Dforce_posix_threads=true \
                        -Dtests=false"
 
-PKG_MESON_PROPERTIES_TARGET="
-have_c99_vsnprintf=false
-have_c99_snprintf=false
-growing_stack=false
-va_val_copy=false"
-
 post_makeinstall_target() {
   rm -rf ${INSTALL}/usr/bin
   rm -rf ${INSTALL}/usr/lib/gdbus-2.0
