diff --git a/packages/lakka/libretro_cores/ppsspp/package.mk b/packages/lakka/libretro_cores/ppsspp/package.mk
index 52b748d3d3..c1f27f7d99 100644
--- a/packages/lakka/libretro_cores/ppsspp/package.mk
+++ b/packages/lakka/libretro_cores/ppsspp/package.mk
@@ -34,12 +34,16 @@ if [ "${VULKAN_SUPPORT}" = "yes" ]; then
   else
     PKG_CMAKE_OPTS_TARGET+=" -DUSE_VULKAN_DISPLAY_KHR=ON -DUSING_X11_VULKAN=OFF"
   fi
+else
+    PKG_CMAKE_OPTS_TARGET+=" -DUSING_X11_VULKAN=OFF"
 fi
 
 if [ "${OPENGL_SUPPORT}" = "no" -a "${OPENGLES_SUPPORT}" = "yes" ]; then
   PKG_CMAKE_OPTS_TARGET+=" -DUSING_GLES2=ON"
 fi
 
+
+
 if [ "${TARGET_ARCH}" = "arm" ]; then
   if [[ "${TARGET_NAME}" =~ "armv8" ]]; then
     PKG_CMAKE_OPTS_TARGET+=" -DFORCED_CPU=armv8"
