diff --git a/packages/lakka/libretro_cores/ppsspp/package.mk b/packages/lakka/libretro_cores/ppsspp/package.mk
index 52b748d3d3..f6b95d8664 100644
--- a/packages/lakka/libretro_cores/ppsspp/package.mk
+++ b/packages/lakka/libretro_cores/ppsspp/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="ppsspp"
-PKG_VERSION="ba0ce344937d17e177ec8656ab957f6b82facdda"
+PKG_VERSION="d6b109b1155c0f0a603f70ff6d61c3db393dec14"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/hrydgard/ppsspp"
 PKG_URL="${PKG_SITE}.git"
