diff --git a/packages/audio/pulseaudio/config/pulseaudio-system.conf b/packages/audio/pulseaudio/config/pulseaudio-system.conf
deleted file mode 100644
index c92fc53010..0000000000
--- a/packages/audio/pulseaudio/config/pulseaudio-system.conf
+++ /dev/null
@@ -1,33 +0,0 @@
-<?xml version="1.0"?><!--*-nxml-*-->
-<!DOCTYPE busconfig PUBLIC "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
- "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
-
-<!--
-This file is part of PulseAudio.
-
-PulseAudio is free software; you can redistribute it and/or modify it
-under the terms of the GNU Lesser General Public License as
-published by the Free Software Foundation; either version 2.1 of the
-License, or (at your option) any later version.
-
-PulseAudio is distributed in the hope that it will be useful, but WITHOUT
-ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
-or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General
-Public License for more details.
-
-You should have received a copy of the GNU Lesser General Public
-License along with PulseAudio; if not, see <http://www.gnu.org/licenses/>.
--->
-
-<busconfig>
-
-  <!-- System-wide PulseAudio runs as 'root' user. This fragment is
-       not necessary for user PulseAudio instances. -->
-
-  <policy user="root">
-    <allow own="org.pulseaudio.Server"/>
-    <allow send_destination="org.pulseaudio.Server"/>
-    <allow receive_sender="org.pulseaudio.Server"/>
-  </policy>
-
-</busconfig>
diff --git a/packages/audio/pulseaudio/config/system.pa b/packages/audio/pulseaudio/config/system.pa
index 7945366058..653361e807 100644
--- a/packages/audio/pulseaudio/config/system.pa
+++ b/packages/audio/pulseaudio/config/system.pa
@@ -19,14 +19,12 @@
 # mode.
 
 ### Load several protocols
-.ifexists module-esound-protocol-unix.so
-load-module module-esound-protocol-unix
-.endif
 load-module module-native-protocol-unix
 
 ### Automatically restore the volume of streams and devices
 load-module module-stream-restore
 load-module module-device-restore
+load-module module-card-restore
 
 ### Automatically restore the default sink/source when changed by the user
 ### during runtime
@@ -34,19 +32,12 @@ load-module module-device-restore
 ### that look up the default sink/source get the right value
 load-module module-default-device-restore
 
-### Automatically move streams to the default sink if the sink they are
-### connected to dies, similar for sources
-load-module module-rescue-streams
-
 ### Make sure we always have a sink around, even if it is a null sink.
 load-module module-always-sink
 
 ### Automatically suspend sinks/sources that become idle for too long
 load-module module-suspend-on-idle
 
-### Enable positioned event sounds
-load-module module-position-event-sounds
-
 ### Automatically load modules for dbus
 .ifexists module-dbus-protocol.so
  load-module module-dbus-protocol
@@ -65,6 +56,4 @@ load-module module-position-event-sounds
   load-module module-zeroconf-publish
 .endif
 
-load-module module-native-protocol-tcp auth-anonymous=1
 load-module module-switch-on-connect
-load-module module-switch-on-port-available
diff --git a/packages/audio/pulseaudio/package.mk b/packages/audio/pulseaudio/package.mk
index 3950cc0493..790cb0797d 100644
--- a/packages/audio/pulseaudio/package.mk
+++ b/packages/audio/pulseaudio/package.mk
@@ -1,123 +1,100 @@
-################################################################################
-#      This file is part of OpenELEC - http://www.openelec.tv
-#      Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
-#
-#  OpenELEC is free software: you can redistribute it and/or modify
-#  it under the terms of the GNU General Public License as published by
-#  the Free Software Foundation, either version 2 of the License, or
-#  (at your option) any later version.
-#
-#  OpenELEC is distributed in the hope that it will be useful,
-#  but WITHOUT ANY WARRANTY; without even the implied warranty of
-#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#  GNU General Public License for more details.
-#
-#  You should have received a copy of the GNU General Public License
-#  along with OpenELEC.  If not, see <http://www.gnu.org/licenses/>.
-################################################################################
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
+# Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="pulseaudio"
-PKG_VERSION="12.99.1"
-PKG_ARCH="any"
+PKG_VERSION="14.2"
+PKG_SHA256="75d3f7742c1ae449049a4c88900e454b8b350ecaa8c544f3488a2562a9ff66f1"
 PKG_LICENSE="GPL"
 PKG_SITE="http://pulseaudio.org/"
-PKG_URL="http://www.freedesktop.org/software/pulseaudio/releases/$PKG_NAME-$PKG_VERSION.tar.xz"
-PKG_DEPENDS_TARGET="toolchain libtool json-c alsa-lib libsndfile soxr dbus systemd openssl libcap"
-PKG_SECTION="audio"
-PKG_SHORTDESC="pulseaudio: Yet another sound server for Unix"
-PKG_LONGDESC="PulseAudio is a sound server for Linux and other Unix-like operating systems. It is intended to be an improved drop-in replacement for the Enlightened Sound Daemon (esound or esd). In addition to the features esound provides, PulseAudio has an extensible plugin architecture, support for more than one sink per source, better low-latency behavior, the ability to be embedded into other software, a completely asynchronous C API, a simple command line interface for reconfiguring the daemon while running, flexible and implicit sample type conversion and resampling, and a "Zero-Copy" architecture."
+PKG_URL="http://www.freedesktop.org/software/pulseaudio/releases/${PKG_NAME}-${PKG_VERSION}.tar.xz"
+PKG_DEPENDS_TARGET="toolchain alsa-lib dbus libcap libsndfile libtool openssl soxr speexdsp systemd glib:host glib"
+PKG_LONGDESC="PulseAudio is a sound system for POSIX OSes, meaning that it is a proxy for your sound applications."
+PKG_TOOLCHAIN="meson"
 
-PKG_IS_ADDON="no"
-
-# broken
-PKG_AUTORECONF="no"
-
-if [ "$BLUETOOTH_SUPPORT" = "yes" ]; then
-  PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET sbc"
-  PULSEAUDIO_BLUETOOTH="--enable-bluez5"
-else
-  PULSEAUDIO_BLUETOOTH="--disable-bluez5"
+if [ "${PROJECT}" = "L4T" ]; then
+  PKG_PATCH_DIRS="${PROJECT}"
 fi
 
-if [ "$AVAHI_DAEMON" = "yes" ]; then
-  PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET avahi"
-  PULSEAUDIO_AVAHI="--enable-avahi"
+if [ "${BLUETOOTH_SUPPORT}" = "yes" ]; then
+  PKG_DEPENDS_TARGET+=" sbc"
+  PKG_PULSEAUDIO_BLUETOOTH="-Dbluez5=true"
 else
-  PULSEAUDIO_AVAHI="--disable-avahi"
+  PKG_PULSEAUDIO_BLUETOOTH="-Dbluez5=false"
 fi
 
-if [ "$TARGET_FPU" = "neon" -o "$TARGET_FPU" = "neon-fp16" -o "$TARGET_FPU" = "neon-vfpv4" ]; then
-  PULSEAUDIO_NEON="--enable-neon-opt"
+if [ "${AVAHI_DAEMON}" = "yes" ]; then
+  PKG_DEPENDS_TARGET+=" avahi"
+  PKG_PULSEAUDIO_AVAHI="-Davahi=enabled"
 else
-  PULSEAUDIO_NEON="--disable-neon-opt"
+  PKG_PULSEAUDIO_AVAHI="-Davahi=disabled"
 fi
 
-# package specific configure options
-PKG_CONFIGURE_OPTS_TARGET="--disable-silent-rules \
-                           --disable-nls \
-                           --enable-largefile \
-                           --disable-rpath \
-                           $PULSEAUDIO_NEON \
-                           --disable-x11 \
-                           --disable-tests \
-                           --disable-samplerate \
-                           --disable-oss-output \
-                           --disable-oss-wrapper \
-                           --disable-coreaudio-output \
-                           --enable-alsa \
-                           --disable-esound \
-                           --disable-solaris \
-                           --disable-waveout \
-                           --enable-glib2 \
-                           --disable-gtk3 \
-                           --disable-gconf \
-                           $PULSEAUDIO_AVAHI \
-                           --disable-jack \
-                           --disable-asyncns \
-                           --disable-tcpwrap \
-                           --disable-lirc \
-                           --enable-dbus \
-                           --disable-bluez4 \
-                           $PULSEAUDIO_BLUETOOTH \
-                           --disable-bluez5-ofono-headset \
-                           --disable-bluez5-native-headset \
-                           --enable-udev \
-                           --with-udev-rules-dir=/usr/lib/udev/rules.d
-                           --disable-hal-compat \
-                           --enable-ipv6 \
-                           --enable-openssl \
-                           --disable-xen \
-                           --disable-orc \
-                           --disable-manpages \
-                           --disable-per-user-esound-socket \
-                           --disable-default-build-tests \
-                           --disable-legacy-database-entry-format \
-                           --with-system-user=root \
-                           --with-system-group=root \
-                           --with-access-group=root \
-                           --without-caps \
-                           --without-fftw \
-                           --without-speex \
-                           --with-soxr \
-                           --with-module-dir=/usr/lib/pulse"
+PKG_MESON_OPTS_TARGET="-Dgcov=false \
+                       -Dman=false \
+                       -Dtests=false \
+                       -Dsystem_user=root \
+                       -Dsystem_group=root \
+                       -Daccess_group=root \
+                       -Ddatabase=simple \
+                       -Dlegacy-database-entry-format=false \
+                       -Drunning-from-build-tree=false \
+                       -Datomic-arm-linux-helpers=true \
+                       -Datomic-arm-memory-barrier=false \
+                       -Dmodlibexecdir=/usr/lib/pulse \
+                       -Dudevrulesdir=/usr/lib/udev/rules.d \
+                       -Dalsa=enabled \
+                       -Dasyncns=disabled \
+                       ${PKG_PULSEAUDIO_AVAHI} \
+                       ${PKG_PULSEAUDIO_BLUETOOTH} \
+                       -Dbluez5-native-headset=false \
+                       -Dbluez5-ofono-headset=false \
+                       -Ddbus=enabled \
+                       -Dfftw=disabled \
+                       -Dglib=enabled \
+                       -Dgsettings=disabled \
+                       -Dgtk=disabled \
+                       -Dhal-compat=false \
+                       -Dipv6=true \
+                       -Djack=disabled \
+                       -Dlirc=disabled \
+                       -Dopenssl=enabled \
+                       -Dorc=disabled \
+                       -Dsamplerate=disabled \
+                       -Dsoxr=enabled \
+                       -Dspeex=enabled \
+                       -Dsystemd=enabled \
+                       -Dudev=enabled \
+                       -Dx11=disabled \
+                       -Dadrian-aec=true \
+                       -Dwebrtc-aec=disabled"
+
+pre_configure_target() {
+  sed -e 's|; remixing-use-all-sink-channels = yes|; remixing-use-all-sink-channels = no|' \
+      -i ${PKG_BUILD}/src/daemon/daemon.conf.in
+}
 
 post_makeinstall_target() {
-  rm -rf $INSTALL/usr/bin/esdcompat
-  rm -rf $INSTALL/usr/include
-  rm -rf $INSTALL/usr/lib/cmake
-  rm -rf $INSTALL/usr/lib/pkgconfig
-  rm -rf $INSTALL/usr/lib/systemd
-  rm -rf $INSTALL/usr/share/vala
-  rm -rf $INSTALL/usr/share/zsh
-  rm -rf $INSTALL/usr/share/bash-completion
+  safe_remove ${INSTALL}/usr/include
+  safe_remove ${INSTALL}/usr/lib/cmake
+  safe_remove ${INSTALL}/usr/lib/pkgconfig
+  safe_remove ${INSTALL}/usr/lib/systemd
+  safe_remove ${INSTALL}/usr/share/vala
+  safe_remove ${INSTALL}/usr/share/zsh
+  safe_remove ${INSTALL}/usr/share/bash-completion
+
+  cp ${PKG_DIR}/config/system.pa ${INSTALL}/etc/pulse/
+
+  if [ "${PROJECT}" = "L4T" ]; then
+    echo load-module module-switch-on-port-available >> ${INSTALL}/etc/pulse/system.pa
+  fi
 
-  cp $PKG_DIR/config/system.pa $INSTALL/etc/pulse/
-  cp $PKG_DIR/config/pulseaudio-system.conf $INSTALL/etc/dbus-1/system.d/
+  sed 's/user="pulse"/user="root"/' -i ${INSTALL}/etc/dbus-1/system.d/pulseaudio-system.conf
 
-  mkdir -p $INSTALL/usr/config
-    cp -PR $PKG_DIR/config/pulse-daemon.conf.d $INSTALL/usr/config
+  mkdir -p ${INSTALL}/usr/config
+    cp -PR ${PKG_DIR}/config/pulse-daemon.conf.d ${INSTALL}/usr/config
 
-  ln -sf /storage/.config/pulse-daemon.conf.d $INSTALL/etc/pulse/daemon.conf.d
+  ln -sf /storage/.config/pulse-daemon.conf.d ${INSTALL}/etc/pulse/daemon.conf.d
 }
 
 post_install() {
diff --git a/packages/audio/pulseaudio/patches/pulseaudio-0900.03-dont_relink_against_host.patch b/packages/audio/pulseaudio/patches/L4T/pulseaudio-0900.03-dont_relink_against_host.patch
similarity index 100%
rename from packages/audio/pulseaudio/patches/pulseaudio-0900.03-dont_relink_against_host.patch
rename to packages/audio/pulseaudio/patches/L4T/pulseaudio-0900.03-dont_relink_against_host.patch
diff --git a/packages/audio/pulseaudio/patches/pulseaudio-0900.01-never-append-dirty-to-server-module-version.patch b/packages/audio/pulseaudio/patches/pulseaudio-100.01-never-append-dirty-to-server-module-version.patch
similarity index 100%
rename from packages/audio/pulseaudio/patches/pulseaudio-0900.01-never-append-dirty-to-server-module-version.patch
rename to packages/audio/pulseaudio/patches/pulseaudio-100.01-never-append-dirty-to-server-module-version.patch
diff --git a/packages/audio/pulseaudio/patches/pulseaudio-0900.02-check_uid.patch b/packages/audio/pulseaudio/patches/pulseaudio-100.02-check_uid.patch
similarity index 79%
rename from packages/audio/pulseaudio/patches/pulseaudio-0900.02-check_uid.patch
rename to packages/audio/pulseaudio/patches/pulseaudio-100.02-check_uid.patch
index a54ede3b8a..df92750890 100644
--- a/packages/audio/pulseaudio/patches/pulseaudio-0900.02-check_uid.patch
+++ b/packages/audio/pulseaudio/patches/pulseaudio-100.02-check_uid.patch
@@ -1,13 +1,13 @@
 --- pulseaudio-4.0.orig/src/pulsecore/core-util.c	2014-01-12 23:31:26.281525000 -0800
 +++ pulseaudio-4.0/src/pulsecore/core-util.c	2014-01-12 23:32:32.977118803 -0800
-@@ -1524,10 +1524,6 @@
+@@ -1447,10 +1447,6 @@
      if (stat(p, &st) < 0)
          return -errno;
  
 -#ifdef HAVE_GETUID
--    if (st.st_uid != getuid())
+-    if (st.st_uid != getuid() && st.st_uid != 0)
 -        return -EACCES;
 -#endif
 
      return 0;
- }
\ No newline at end of file
+ }
diff --git a/packages/audio/pulseaudio/patches/pulseaudio-12.99.3-01-alsa2-ucm-fix.patch b/packages/audio/pulseaudio/patches/pulseaudio-12.99.3-01-alsa2-ucm-fix.patch
deleted file mode 100644
index 9219b57d0e..0000000000
--- a/packages/audio/pulseaudio/patches/pulseaudio-12.99.3-01-alsa2-ucm-fix.patch
+++ /dev/null
@@ -1,61 +0,0 @@
-From c8f065250dde966825f171ff817f7301f423a42e Mon Sep 17 00:00:00 2001
-From: Jaroslav Kysela <perex@perex.cz>
-Date: Sat, 23 Nov 2019 15:17:30 +0100
-Subject: [PATCH] alsa-ucm: use ucm2 name for the direct card index open
-
-Signed-off-by: Jaroslav Kysela <perex@perex.cz>
----
- src/modules/alsa/alsa-ucm.c | 28 ++++++++++++++++++----------
- 1 file changed, 18 insertions(+), 10 deletions(-)
-
-diff --git a/src/modules/alsa/alsa-ucm.c b/src/modules/alsa/alsa-ucm.c
-index 0a40ca8fe..aeb4e59e3 100644
---- a/src/modules/alsa/alsa-ucm.c
-+++ b/src/modules/alsa/alsa-ucm.c
-@@ -576,17 +576,25 @@ int pa_alsa_ucm_query_profiles(pa_alsa_ucm_config *ucm, int card_index) {
-     const char **verb_list;
-     int num_verbs, i, err = 0;
- 
--    /* is UCM available for this card ? */
--    err = snd_card_get_name(card_index, &card_name);
--    if (err < 0) {
--        pa_log("Card can't get card_name from card_index %d", card_index);
--        goto name_fail;
--    }
--
-+    /* support multiple card instances, address card directly by index */
-+    card_name = pa_sprintf_malloc("hw:%i", card_index);
-+    if (card_name == NULL)
-+        return -ENOMEM;
-     err = snd_use_case_mgr_open(&ucm->ucm_mgr, card_name);
-     if (err < 0) {
--        pa_log_info("UCM not available for card %s", card_name);
--        goto ucm_mgr_fail;
-+        /* fallback longname: is UCM available for this card ? */
-+        pa_xfree(card_name);
-+        err = snd_card_get_name(card_index, &card_name);
-+        if (err < 0) {
-+            pa_log("Card can't get card_name from card_index %d", card_index);
-+            goto name_fail;
-+        }
-+
-+        err = snd_use_case_mgr_open(&ucm->ucm_mgr, card_name);
-+        if (err < 0) {
-+            pa_log_info("UCM not available for card %s", card_name);
-+            goto ucm_mgr_fail;
-+        }
-     }
- 
-     pa_log_info("UCM available for card %s", card_name);
-@@ -626,7 +634,7 @@ ucm_verb_fail:
-     }
- 
- ucm_mgr_fail:
--    free(card_name);
-+    pa_xfree(card_name);
- 
- name_fail:
-     return err;
--- 
-GitLab
-
diff --git a/packages/audio/pulseaudio/patches/pulseaudio-12.99.3-02-alsa2-ucm-fix.patch b/packages/audio/pulseaudio/patches/pulseaudio-12.99.3-02-alsa2-ucm-fix.patch
deleted file mode 100644
index 886541d0ce..0000000000
--- a/packages/audio/pulseaudio/patches/pulseaudio-12.99.3-02-alsa2-ucm-fix.patch
+++ /dev/null
@@ -1,124 +0,0 @@
-From 3549a4d926c9505f3600a8eb7941dc27a9039b3f Mon Sep 17 00:00:00 2001
-From: Hui Wang <hui.wang@canonical.com>
-Date: Fri, 8 May 2020 11:19:48 +0800
-Subject: [PATCH] alsa-mixer: store the ucm_device with the order of their
- priority
-
-There is some case that multiple ucm devices share an amixer Jack
-like "Headphones", "Headset" and "Mic2" share the "Headphone Mic Jack",
-When the Jack state is changed, the module-switch-on-port-available
-will process them in the order they are in the jack->ucm_devices, and
-the last device will decide the final setting.
-
-But usually users put priority for those devices and expect the
-final setting is based on the highest priority device if there is no
-other policies like manual selection. So here do some change to store
-the ucm_devices according to their priority (from low to high).
-
-For example, we have ucm devices definition like below (ucm2):
-               SectionDevice."Mic2" {
-                        Comment "Headphones Stereo Microphone"
-			...
-                        Value {
-                                CapturePriority 200
-				...
-                }
-
-                SectionDevice."Headset" {
-                        Comment "Headset Mono Microphone"
-			...
-                        Value {
-                                CapturePriority 300
-				...
-                        }
-                }
-
-Without this patch, the final setting is based on Mic2, after applying
-this patch, the final setting is based on the Headset (with higher
-priority than Mic2).
-
-Signed-off-by: Hui Wang <hui.wang@canonical.com>
----
- src/modules/alsa/alsa-mixer.c | 15 ++++++++++++++-
- src/pulsecore/dynarray.c      | 23 +++++++++++++++++++++++
- src/pulsecore/dynarray.h      |  4 ++++
- 3 files changed, 41 insertions(+), 1 deletion(-)
-
-diff --git a/src/modules/alsa/alsa-mixer.c b/src/modules/alsa/alsa-mixer.c
-index d184aec7a..f2fe20057 100644
---- a/src/modules/alsa/alsa-mixer.c
-+++ b/src/modules/alsa/alsa-mixer.c
-@@ -249,10 +249,23 @@ void pa_alsa_jack_set_plugged_in(pa_alsa_jack *jack, bool plugged_in) {
- }
- 
- void pa_alsa_jack_add_ucm_device(pa_alsa_jack *jack, pa_alsa_ucm_device *device) {
-+    pa_alsa_ucm_device *idevice;
-+    unsigned idx, prio, iprio;
-+
-     pa_assert(jack);
-     pa_assert(device);
- 
--    pa_dynarray_append(jack->ucm_devices, device);
-+    /* store the ucm device with the sequence of priority from low to high. this
-+     * could guarantee when the jack state is changed, the device with highest
-+     * priority will send to the module-switch-on-port-available last */
-+    prio = device->playback_priority ? device->playback_priority : device->capture_priority;
-+
-+    PA_DYNARRAY_FOREACH(idevice, jack->ucm_devices, idx) {
-+        iprio = idevice->playback_priority ? idevice->playback_priority : idevice->capture_priority;
-+        if (iprio > prio)
-+            break;
-+    }
-+    pa_dynarray_insert_by_index(jack->ucm_devices, device, idx);
- }
- 
- void pa_alsa_jack_add_ucm_hw_mute_device(pa_alsa_jack *jack, pa_alsa_ucm_device *device) {
-diff --git a/src/pulsecore/dynarray.c b/src/pulsecore/dynarray.c
-index 6a3eb5f04..a948f2671 100644
---- a/src/pulsecore/dynarray.c
-+++ b/src/pulsecore/dynarray.c
-@@ -140,3 +140,26 @@ unsigned pa_dynarray_size(pa_dynarray *array) {
- 
-     return array->n_entries;
- }
-+
-+int pa_dynarray_insert_by_index(pa_dynarray *array, void *p, unsigned i) {
-+    void *entry;
-+    unsigned j;
-+
-+    pa_assert(array);
-+
-+    if (i > array->n_entries)
-+        return -PA_ERR_NOENTITY;
-+
-+    if (i == array->n_entries)
-+        pa_dynarray_append(array, p);
-+    else {
-+        entry = pa_dynarray_last(array);
-+        pa_dynarray_append(array, entry);
-+        j = array->n_entries - 2;
-+        for (;j > i; j--)
-+	    array->data[j] = array->data[j-1];
-+        array->data[i] = p;
-+    }
-+
-+    return 0;
-+}
-diff --git a/src/pulsecore/dynarray.h b/src/pulsecore/dynarray.h
-index bf7dddc12..4c0925e7c 100644
---- a/src/pulsecore/dynarray.h
-+++ b/src/pulsecore/dynarray.h
-@@ -63,6 +63,10 @@ void *pa_dynarray_steal_last(pa_dynarray *array);
- 
- unsigned pa_dynarray_size(pa_dynarray *array);
- 
-+/* Returns -PA_ERR_NOENTITY if i is out of bounds, and zero otherwise.
-+ * Here i is the location index in the array like 0, ..., array->entries */
-+int pa_dynarray_insert_by_index(pa_dynarray *array, void *p, unsigned i);
-+
- #define PA_DYNARRAY_FOREACH(elem, array, idx) \
-     for ((idx) = 0; ((elem) = pa_dynarray_get(array, idx)); (idx)++)
- 
--- 
-GitLab
-
diff --git a/packages/audio/pulseaudio/patches/pulseaudio-12.99.3-03-alsa2-ucm-fix.patch b/packages/audio/pulseaudio/patches/pulseaudio-12.99.3-03-alsa2-ucm-fix.patch
deleted file mode 100644
index 2ad453a909..0000000000
--- a/packages/audio/pulseaudio/patches/pulseaudio-12.99.3-03-alsa2-ucm-fix.patch
+++ /dev/null
@@ -1,89 +0,0 @@
-From c8653c13faed5d877db3870335b870c144f01513 Mon Sep 17 00:00:00 2001
-From: Hui Wang <hui.wang@canonical.com>
-Date: Thu, 11 Jun 2020 15:49:05 +0800
-Subject: [PATCH] alsa: adjust ucm sink/source priority according to ports
- priority
-
-On the machines with the ucm used, the different input/output devices
-often have different pcm stream, so they often belong to different
-sources and sinks, this is greatly different from the design of all
-devices connected to a codec (without ucm).
-
-For example, on a machine with ucm2 used:
-the internal dmic is on source#0
-the external mic is on the source#1
-the internal spk is on sink#0
-the external headphone is on sink#1
-
-Users expect that after plugging the external device, it will become
-the active device automatically. The switch-on-port-available could
-make it to be the active_port on its own source/sink, but can't make
-source/sink to be default_source/sink since the sources/sinks belong
-to the same profile (HiFi usually).
-
-If we adjust the source/sink priority according to ucm ports priority,
-the device_port.c could handle the default_source/sink changing then.
-Usually we set higher priority for external device than internal
-device in the ucm.
-
-In order to bring the lowest side effect on the source/sink priority,
-I change the ucm priority to units digit first, then add it to the
-original priority.
-
-Signed-off-by: Hui Wang <hui.wang@canonical.com>
----
- src/modules/alsa/alsa-sink.c   | 13 +++++++++++++
- src/modules/alsa/alsa-source.c | 13 +++++++++++++
- 2 files changed, 26 insertions(+)
-
-diff --git a/src/modules/alsa/alsa-sink.c b/src/modules/alsa/alsa-sink.c
-index 457dc2785..3fe429c80 100644
---- a/src/modules/alsa/alsa-sink.c
-+++ b/src/modules/alsa/alsa-sink.c
-@@ -2598,6 +2598,19 @@ pa_sink *pa_alsa_sink_new(pa_module *m, pa_modargs *ma, const char*driver, pa_ca
-         goto fail;
-     }
- 
-+    if (u->ucm_context) {
-+        pa_device_port *port;
-+        void *state;
-+        unsigned h_prio = 0;
-+        PA_HASHMAP_FOREACH(port, u->sink->ports, state) {
-+            if (!h_prio || port->priority > h_prio)
-+                h_prio = port->priority;
-+        }
-+        /* ucm ports prioriy is 100, 200, ..., 900, change it to units digit */
-+        h_prio = h_prio / 100;
-+        u->sink->priority += h_prio;
-+    }
-+
-     if (pa_modargs_get_value_u32(ma, "deferred_volume_safety_margin",
-                                  &u->sink->thread_info.volume_change_safety_margin) < 0) {
-         pa_log("Failed to parse deferred_volume_safety_margin parameter");
-diff --git a/src/modules/alsa/alsa-source.c b/src/modules/alsa/alsa-source.c
-index da99ac610..b1149c675 100644
---- a/src/modules/alsa/alsa-source.c
-+++ b/src/modules/alsa/alsa-source.c
-@@ -2284,6 +2284,19 @@ pa_source *pa_alsa_source_new(pa_module *m, pa_modargs *ma, const char*driver, p
-         goto fail;
-     }
- 
-+    if (u->ucm_context) {
-+        pa_device_port *port;
-+        void *state;
-+        unsigned h_prio = 0;
-+        PA_HASHMAP_FOREACH(port, u->source->ports, state) {
-+            if (!h_prio || port->priority > h_prio)
-+                h_prio = port->priority;
-+        }
-+        /* ucm ports prioriy is 100, 200, ..., 900, change it to units digit */
-+        h_prio = h_prio / 100;
-+        u->source->priority += h_prio;
-+    }
-+
-     if (pa_modargs_get_value_u32(ma, "deferred_volume_safety_margin",
-                                  &u->source->thread_info.volume_change_safety_margin) < 0) {
-         pa_log("Failed to parse deferred_volume_safety_margin parameter");
--- 
-GitLab
-
diff --git a/packages/audio/pulseaudio/profile.d/99-pulseaudio.conf b/packages/audio/pulseaudio/profile.d/99-pulseaudio.conf
index 70427c6cf2..7e7b95357b 100644
--- a/packages/audio/pulseaudio/profile.d/99-pulseaudio.conf
+++ b/packages/audio/pulseaudio/profile.d/99-pulseaudio.conf
@@ -1,19 +1,4 @@
-################################################################################
-#      This file is part of OpenELEC - http://www.openelec.tv
-#      Copyright (C) 2009-2014 Stephan Raue (stephan@openelec.tv)
-#
-#  OpenELEC is free software: you can redistribute it and/or modify
-#  it under the terms of the GNU General Public License as published by
-#  the Free Software Foundation, either version 2 of the License, or
-#  (at your option) any later version.
-#
-#  OpenELEC is distributed in the hope that it will be useful,
-#  but WITHOUT ANY WARRANTY; without even the implied warranty of
-#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#  GNU General Public License for more details.
-#
-#  You should have received a copy of the GNU General Public License
-#  along with OpenELEC.  If not, see <http://www.gnu.org/licenses/>.
-################################################################################
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2009-2014 Stephan Raue (stephan@openelec.tv)
 
 export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/lib/pulseaudio"
diff --git a/packages/audio/pulseaudio/system.d/pulseaudio.service b/packages/audio/pulseaudio/system.d/pulseaudio.service
index f8da4589b1..1ec4269550 100644
--- a/packages/audio/pulseaudio/system.d/pulseaudio.service
+++ b/packages/audio/pulseaudio/system.d/pulseaudio.service
@@ -1,18 +1,14 @@
 [Unit]
-Description=PulseAudio Sound System
+Description=PulseAudio Sound Service
 After=syslog.target local-fs.target
 
 [Service]
+Type=dbus
+BusName=org.pulseaudio.Server
+ExecStart=/usr/bin/pulseaudio --system --disallow-exit --exit-idle-time=-1 --disable-shm --enable-memfd --realtime
 Restart=on-failure
-ExecStartPre=-/bin/sh -c "rm -rf /storage/.config/pulse"
-ExecStartPre=-/bin/sh -c "mkdir -p /storage/.config/pulse"
-ExecStartPre=-/bin/sh -c "mkdir -p /var/pulseconfig"
-ExecStartPre=-/bin/sh -c "mount --bind /var/pulseconfig /storage/.config/pulse"
-ExecStart=/usr/bin/pulseaudio --system
-TimeoutStopSec=1s
-RestartSec=2
-StartLimitInterval=60
-StartLimitBurst=10
+Environment=PULSE_STATE_PATH=/storage/.config/pulse
+Environment=PULSE_CONFIG_PATH=/storage/.config/pulse
 
 [Install]
 WantedBy=multi-user.target
diff --git a/packages/audio/speexdsp/package.mk b/packages/audio/speexdsp/package.mk
new file mode 100644
index 0000000000..18accd9a2c
--- /dev/null
+++ b/packages/audio/speexdsp/package.mk
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="speexdsp"
+PKG_VERSION="1.2.0"
+PKG_SHA256="682042fc6f9bee6294ec453f470dadc26c6ff29b9c9e9ad2ffc1f4312fd64771"
+PKG_LICENSE="BSD"
+PKG_SITE="https://speex.org"
+PKG_URL="http://downloads.us.xiph.org/releases/speex/speexdsp-${PKG_VERSION}.tar.gz"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_LONGDESC="Speex audio processing library"
