diff --git a/packages/libretro/scummvm/package.mk b/packages/libretro/scummvm/package.mk
index 37b15e3023..7b7ee45b34 100644
--- a/packages/libretro/scummvm/package.mk
+++ b/packages/libretro/scummvm/package.mk
@@ -19,7 +19,7 @@
 ################################################################################
 
 PKG_NAME="scummvm"
-PKG_VERSION="63e5757"
+PKG_VERSION="80cb726"
 PKG_ARCH="any"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/libretro/scummvm"

