diff --git a/packages/libretro/play/package.mk b/packages/libretro/play/package.mk
index aee751a9b7..bf7d508985 100644
--- a/packages/libretro/play/package.mk
+++ b/packages/libretro/play/package.mk
@@ -19,7 +19,7 @@
 ################################################################################
 
 PKG_NAME="play"
-PKG_VERSION="a37d98b"
+PKG_VERSION="30a3978"
 PKG_ARCH="any"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/jpd002/Play-"
