diff --git a/packages/lakka/retroarch_base/retroarch/package.mk b/packages/lakka/retroarch_base/retroarch/package.mk
index 4bd4c11efa..518b944f0c 100644
--- a/packages/lakka/retroarch_base/retroarch/package.mk
+++ b/packages/lakka/retroarch_base/retroarch/package.mk
@@ -187,6 +187,7 @@ makeinstall_target() {
   echo 'log_dir = "/storage/logfiles"' >> ${INSTALL}/etc/retroarch.cfg
   echo 'recording_output_directory = "/storage/recordings"' >> ${INSTALL}/etc/retroarch.cfg
   echo 'menu_driver = "xmb"' >> ${INSTALL}/etc/retroarch.cfg
+  echo 'xmb_alpha_factor = "100"' >> ${INSTALL}/etc/retroarch.cfg
 
   # Power settings
   # Use ondemand for all RPi devices (for backwards compatibility?)
