diff --git a/packages/lakka/libretro_cores/flycast/package.mk b/packages/lakka/libretro_cores/flycast/package.mk
index aa5823ff07..009f88f33f 100644
--- a/packages/lakka/libretro_cores/flycast/package.mk
+++ b/packages/lakka/libretro_cores/flycast/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="flycast"
-PKG_VERSION="8e96590065fd23cd37a0d426d30f4aef6587d860"
+PKG_VERSION="8a0323ec087f8bf85a419ac7ced575f6065d5758"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/flyinghead/flycast"
 PKG_URL="${PKG_SITE}.git"
