diff --git a/packages/lakka/libretro_cores/flycast/package.mk b/packages/lakka/libretro_cores/flycast/package.mk
index 330706cd8c..1fbaa46f5e 100644
--- a/packages/lakka/libretro_cores/flycast/package.mk
+++ b/packages/lakka/libretro_cores/flycast/package.mk
@@ -1,5 +1,5 @@
 PKG_NAME="flycast"
-PKG_VERSION="a5967a283f65ae12f31765ee27a70ec82073a226"
+PKG_VERSION="8a0323ec087f8bf85a419ac7ced575f6065d5758"
 PKG_LICENSE="GPLv2"
 PKG_SITE="https://github.com/flyinghead/flycast"
 PKG_URL="${PKG_SITE}.git"
